diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 707798bb079977281e8a8642cb6a0d7ccf214f5e..e9f8679789c086cbd8079a8f809a127ac88100bd 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -341,6 +341,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
