diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 9b1b6c6e5a1c45daa50b38d53084e7d2127c01be..31cc5e5dc057f46be4368e16bc0043cb35be65ad 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -336,6 +336,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
