diff --git a/base/BUILD.gn b/base/BUILD.gn
index ff71625d3745fe7a667642bb06579febb5ea4a6b..af9e89ba833516210adae16c31fbd7db3c97b037 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4455,6 +4455,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4493,6 +4494,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_aidl("process_launcher_aidl") {
