diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 512b74ea492907219196e9a135f1f9288d346307..e212da91b954e50c1651d2072cd44d3adf8735e6 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -346,6 +346,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
