diff --git a/base/BUILD.gn b/base/BUILD.gn
index 554604d2cb9f076987fa4581aa072ffa2bd9a2b0..fba30ed4a72057d5ab245d7ef4a00f832d5abb04 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4129,6 +4129,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
