diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index daf4ce0c061a04dbb7433a224c4db6e7b42afa0b..6811b044b4ea980ca8f413930838bf122d72435d 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -339,6 +339,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
