diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 7b9ce0172eedf2b55fe7f3de7baee2f2028f4a7b..2ec432e30bb704f5f19d894fe05b926823356dc5 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -735,6 +735,7 @@ config("compiler") {
     if (current_cpu == "arm") {
       ldflags += [ "-march=$arm_arch" ]
     }
+    ldflags -= brave_removed_compiler_thinlto_flags
   }
 
   if (compiler_timing) {
@@ -1681,6 +1682,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
