diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index 83c6c800344354794d44b49049529333d4feb349..ccce25350db7c10a9cfdaace8fa256c942c3d508 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -362,6 +362,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
