diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 28f6822d85721e832b2841e11e32183ca008c093..ea6457a56bcd6faadc461e74651d7fc50ea985f5 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -557,7 +557,6 @@ config("compiler") {
   }
 
   if (is_clang && !is_nacl) {
-    cflags += [ "-fcrash-diagnostics-dir=" + clang_diagnostic_dir ]
     if (save_reproducers_on_lld_crash && use_lld) {
       ldflags += [
         "-fcrash-diagnostics=all",
