diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index fa0de909a056f39e7a9b1fc55ce1207d6686e9a9..19937eb749e61334d03965af30a8c1fe05755b35 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -25,6 +25,7 @@ robolectric_toolchain = "//build/toolchain/android:robolectric_$host_cpu"
 # toolchain, this GNI file may be read and processed from within Chrome OS
 # toolchains. Checking |is_android| here would therefore be too restrictive.
 if (is_android || is_chromeos) {
+  import("//brave/build/android/config.gni")
   import("//build/config/android/channel.gni")
   import("//build/config/clang/clang.gni")
   import("//build/config/dcheck_always_on.gni")
