diff --git a/build/config/rust.gni b/build/config/rust.gni
index deafd5e6602057bda325464dc26980cbb9f50052..e0e96aa99854ca32afe393da146f8a1aefc4bed1 100644
--- a/build/config/rust.gni
+++ b/build/config/rust.gni
@@ -130,6 +130,8 @@ if (toolchain_has_rust && rust_sysroot_absolute == "") {
     # Android toolchain version.
     rustc_revision = "rustc 1.64.0-dev (Android Rust Toolchain version 9099361)"
   }
+} else {
+  rustc_revision = "rustc 1.67.0 (fc594f156 2023-01-24)"
 }
 
 # TODO(crbug.com/1278030): To build unit tests for Android we need to build
