diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index ae13db7c3748250c83b152927f5d217ce1fe8fcc..896a18a63e735ac4acfa5c8fb17692df5b9efa97 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -596,7 +596,6 @@ config("compiler") {
   }
 
   if (is_clang && !is_nacl && current_os != "zos") {
-    cflags += [ "-fcrash-diagnostics-dir=" + clang_diagnostic_dir ]
     if (save_reproducers_on_lld_crash && use_lld) {
       ldflags += [
         "-fcrash-diagnostics=all",
