diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index e45fa2c80b97c1512771b53af474751d77395983..cf5906c700a9e03929e4c1e37c397427667c36d3 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1630,6 +1630,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
