diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 92688d1c634626fbf7bd0157dc420a3747735a13..19479104cdf8197604c3b8cd0806bf7f9b4730a6 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1583,6 +1583,7 @@ source_set("extensions") {
       "//components/enterprise/data_controls/core/browser",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 proto_library("cws_item_service_proto") {
