diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index c96bd7218659fca8e0d46cc86115d192ca535be6..1ec561d4ccdd212d30c2410e3f53512d67f610b7 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1511,6 +1511,7 @@ source_set("extensions") {
       "//components/enterprise/data_controls",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 proto_library("cws_item_service_proto") {
