diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index c9ff70a8f2f718de759c1c5c6d6b598ecd62a695..dd6837ad075c9e3ea35821157705c77abc9008dd 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2782,6 +2782,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
