diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index 400e11eb2e677e2a6312956b8c297f8ab881de0d..23713e374bde27f1dbe7eb8ddf56ce4cffd573b8 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -9,6 +9,7 @@ import {pageVisibility} from './page_visibility.js';
 import type {SettingsRoutes} from './router.js';
 import {Route, Router} from './router.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -310,6 +311,7 @@ function createRoutes(): SettingsRoutes {
     }
     // </if>
   }
+  addBraveRoutes(r);
   return r as unknown as SettingsRoutes;
 }
 
