diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 29c0f04be02225524849c447ab089d9d4290cf36..69d7f4b6d05799afc1406ac0c5722329c8dc146a 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4134,6 +4134,7 @@ const FeatureEntry::FeatureVariation kClayBlockingVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
