diff --git a/chrome/browser/extensions/chrome_process_manager_delegate.cc b/chrome/browser/extensions/chrome_process_manager_delegate.cc
index e79841d929b54d4d919aaa34188d0c61e024e65e..06fb3d3d84ba3e96a61a6301423ddbfa1c35f248 100644
--- a/chrome/browser/extensions/chrome_process_manager_delegate.cc
+++ b/chrome/browser/extensions/chrome_process_manager_delegate.cc
@@ -164,6 +164,7 @@ void ChromeProcessManagerDelegate::OnBrowserAdded(Browser* browser) {
     Profile* original_profile = profile->GetOriginalProfile();
     ProcessManager* original_manager = ProcessManager::Get(original_profile);
     DCHECK(original_manager);
+    BRAVE_ON_BROWSER_OPENED
     DCHECK_EQ(original_profile, original_manager->browser_context());
     original_manager->MaybeCreateStartupBackgroundHosts();
   }
