diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index f1929da7b0c54281bb3287153e5d3442d41b55b8..5cdbf16837cb27539a80e80ef12c357a85bd9d76 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2516,6 +2516,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
