diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 0fbff7f260878e3b7efad03d28e2474d710d9ef2..1ef17d55fdc8e972ed0d984b8b1e4c38831dc614 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -459,6 +459,7 @@ void ToolbarView::Init() {
           browser_view_, browser_->profile()));
 #endif
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = container_view_->AddChildView(
       std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
