diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index a69e2839ecac74fc121f0c67f7a01312c295f823..c621a8c80cb4644af65209a538f65d9dba4f8ee0 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -514,6 +514,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
