diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java
index 1bfe66d5bba7752f27283ad86b0d5eb8ad0a3f18..5993e56e12bd4fe27d020d1ec0bae0c485f75b04 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java
@@ -214,7 +214,7 @@ public class NotificationServiceImpl extends NotificationService.Impl {
                                 .recordIntentHandlerJobStage(
                                         NotificationUmaTracker.IntentHandlerJobStage.DISPATCH_EVENT,
                                         intent.getAction());
-                        if (!NotificationPlatformBridge.dispatchNotificationEvent(intent)) {
+                        if (!BraveNotificationPlatformBridge.dispatchNotificationEvent(intent)) {
                             Log.w(TAG, "Unable to dispatch the notification event to Chrome.");
                         }
 
