diff --git a/chrome/browser/profiles/profile_impl.h b/chrome/browser/profiles/profile_impl.h
index 28315594f50c07150a1d29a5546cc7359a93cbb2..0aa8b465de7c136480c98db01ec83f6b82b14fa2 100644
--- a/chrome/browser/profiles/profile_impl.h
+++ b/chrome/browser/profiles/profile_impl.h
@@ -32,6 +32,7 @@
 
 class MediaDeviceIDSalt;
 class PrefService;
+class ProfileManager;
 
 #if defined(OS_CHROMEOS)
 namespace chromeos {
@@ -179,6 +180,7 @@ class ProfileImpl : public Profile {
 #endif  // defined(OS_CHROMEOS)
 
  private:
+  friend class ProfileManager;
 #if defined(OS_CHROMEOS)
   friend class chromeos::KioskTest;
   friend class chromeos::SupervisedUserTestBase;
