diff --git a/chrome/browser/ui/views/profiles/profile_menu_view_base.h b/chrome/browser/ui/views/profiles/profile_menu_view_base.h
index c752c7066e152c1a3f73b2b37dc8a5f34366c715..980a90bc7cf834827d680aa416699274f034063c 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view_base.h
+++ b/chrome/browser/ui/views/profiles/profile_menu_view_base.h
@@ -164,6 +164,7 @@ class ProfileMenuViewBase : public content::WebContentsDelegate,
   void set_perform_menu_actions_for_testing(bool perform_menu_actions) {
     perform_menu_actions_ = perform_menu_actions;
   }
+  BRAVE_PROFILE_MENU_VIEW_BASE_H_
 
  private:
   class AXMenuWidgetObserver;
