diff --git a/chrome/browser/resources/tools/build_webui.gni b/chrome/browser/resources/tools/build_webui.gni
index 4d3dcda0e79d6648fd45aed743e49171d7a23c70..743a2ff487d39c4d424bdb525a0acdd3fc3aebb3 100644
--- a/chrome/browser/resources/tools/build_webui.gni
+++ b/chrome/browser/resources/tools/build_webui.gni
@@ -132,6 +132,9 @@ template("build_webui") {
     if (defined(invoker.css_files)) {
       in_files += invoker.css_files
     }
+
+    if (defined(invoker.exclude_preprocess_files)) { in_files -= invoker.exclude_preprocess_files }
+    if (defined(invoker.preprocess_deps)) { public_deps = invoker.preprocess_deps }
   }
 
   if (defined(html_files)) {
