diff --git a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
index cfe79bae17e88dd4379f8ddbdbb288779b51e485..ff31fb181c7ee42b4b4c68a15e114d1185eece01 100644
--- a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
+++ b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
@@ -343,6 +343,7 @@ void RecentTabsSubMenuModel::Build() {
   }
   AddSeparator(ui::NORMAL_SEPARATOR);
   history_separator_index_ = GetItemCount() - 1;
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_BUILD
   BuildLocalEntries();
   BuildTabsFromOtherDevices();
 }
@@ -444,6 +445,7 @@ void RecentTabsSubMenuModel::BuildTabsFromOtherDevices() {
     DCHECK(!session->GetSessionName().empty());
     std::unique_ptr<ui::SimpleMenuModel> device_menu_model =
         CreateOtherDeviceSubMenu(session, tabs_in_session);
+    BRAVE_RECENT_TABS_SUB_MENU_MODEL_BUILD_TABS_FROM_OTHER_DEVICES
     const int command_id = GetAndIncrementNextMenuID();
     AddSubMenu(command_id, base::UTF8ToUTF16(session->GetSessionName()),
                device_menu_model.get());
