diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 31f2a8718d56284e2b848d0b6bf2c5f5f17ee484..ffb9ca82b9223cab6845d00b0a33420a714977d6 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3965,6 +3965,7 @@ const flags_ui::FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
