diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 12530139caf8f0f34dd6cd096597583ae9fb7a36..d835703b1c4a16e555a305893e8d6ff3d5e31a90 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2773,6 +2773,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
