diff --git a/chrome/browser/web_applications/os_integration/web_app_shortcut_mac.mm b/chrome/browser/web_applications/os_integration/web_app_shortcut_mac.mm
index 500a134753e87847f6d2448095397d12b388828a..bb95d8ba75b3978a2098ea8976340608d1dad756 100644
--- a/chrome/browser/web_applications/os_integration/web_app_shortcut_mac.mm
+++ b/chrome/browser/web_applications/os_integration/web_app_shortcut_mac.mm
@@ -1162,6 +1162,7 @@ base::FilePath GetChromeAppsFolder() {
   if (path.empty())
     return path;
 
+  return path.Append(GetLocalizableBraveAppShortcutsSubdirName());
   return path.Append(GetLocalizableAppShortcutsSubdirName());
 }
 
