diff --git a/chrome/browser/bookmarks/managed_bookmark_service_factory.cc b/chrome/browser/bookmarks/managed_bookmark_service_factory.cc
index ce25a3ebc68f82897a2072f1f3ee20cd14598418..d4506342709fbb2dff00072eced89d2aab6c1213 100644
--- a/chrome/browser/bookmarks/managed_bookmark_service_factory.cc
+++ b/chrome/browser/bookmarks/managed_bookmark_service_factory.cc
@@ -73,6 +73,7 @@ KeyedService* ManagedBookmarkServiceFactory::BuildServiceInstanceFor(
 
 content::BrowserContext* ManagedBookmarkServiceFactory::GetBrowserContextToUse(
     content::BrowserContext* context) const {
+  BRAVE_GET_BROWSER_CONTEXT_TO_USE
   return chrome::GetBrowserContextRedirectedInIncognito(context);
 }
 
