diff --git a/chrome/browser/resources/history/history.ts b/chrome/browser/resources/history/history.ts
index 5f2f66549743c24a99ba16c61dbda530cbfee58a..9a548a926ea8b4aab0f9ba1b84ebdaf945422fd0 100644
--- a/chrome/browser/resources/history/history.ts
+++ b/chrome/browser/resources/history/history.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './app.js';
 
 export {BrowserProxyImpl} from 'chrome://resources/cr_components/history_clusters/browser_proxy.js';
