diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
index 08fc569a34ee4f25182ae58e97dd1bede9ae6b72..6f1d2a25371c6246604220474c2739e4ff16c03e 100644
--- a/chrome/test/base/testing_profile.cc
+++ b/chrome/test/base/testing_profile.cc
@@ -417,7 +417,7 @@ void TestingProfile::Init() {
       EnsureBrowserContextKeyedServiceFactoriesBuilt();
 
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
-  if (!IsOffTheRecord()) {
+  if (!IsOffTheRecord() && !brave::IsTorProfile(GetPath())) {
     SupervisedUserSettingsService* settings_service =
         SupervisedUserSettingsServiceFactory::GetForKey(key_.get());
     TestingPrefStore* store = new TestingPrefStore();
