diff --git a/chrome/browser/profiles/profile_impl.cc b/chrome/browser/profiles/profile_impl.cc
index 71619ca3d5271b0f813110dce0e5570b6eebdda7..c53952bc42ed98e9e721d0f74028d2ebe0165c76 100644
--- a/chrome/browser/profiles/profile_impl.cc
+++ b/chrome/browser/profiles/profile_impl.cc
@@ -508,6 +508,9 @@ ProfileImpl::ProfileImpl(
   LoadPrefsForNormalStartup(async_prefs);
 #endif
 
+// Done here instead of browser_prefs.cc so it can override default prefs
+brave::RegisterProfilePrefs(pref_registry_.get());
+
   content::BrowserContext::Initialize(this, path_);
 
   // Register on BrowserContext.
