diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 1dbffe7830002d9b686dcfbcab13fa9b162d2fd2..fbf5ba3f8be287e471f3b931680e03e505e8616d 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -98,6 +98,7 @@ class AvatarToolbarButton : public ToolbarButton {
   static void SetIPHMinDelayAfterCreationForTesting(base::TimeDelta delay);
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
