diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java
index 30f40a91f1a2a3e72352f77f90b3300f1c4ca8ff..5303c5b4ca20fa9360ebc49643aea73b450c3947 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationServiceImpl.java
@@ -213,7 +213,7 @@ public class NotificationServiceImpl extends NotificationService.Impl {
                                 .recordIntentHandlerJobStage(
                                         NotificationUmaTracker.IntentHandlerJobStage.DISPATCH_EVENT,
                                         intent.getAction());
-                        if (!NotificationPlatformBridge.dispatchNotificationEvent(intent)) {
+                        if (!BraveNotificationPlatformBridge.dispatchNotificationEvent(intent)) {
                             Log.w(TAG, "Unable to dispatch the notification event to Chrome.");
                         }
 
