diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index 833607b48d04d769c50c2d94fc16f93bc0e22207..ecb1257b85d51af33e4b184928aaf6b6766b6d26 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -530,6 +530,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
   std::vector<gfx::Rect> CalculateBoundsForDraggedViews(
       const std::vector<raw_ptr<TabSlotView, VectorExperimental>>& views)
       override {
+    BRAVE_TAB_DRAG_CONTEXT_IMPL_CALCULATE_BOUNDS_FOR_DRAGGED_VIEWS
     CHECK(!views.empty(), base::NotFatalUntil::M128)
         << "The views vector must not be empty.";
 
@@ -727,6 +728,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
     // index.
     std::stable_sort(orderable_children.begin(), orderable_children.end());
 
+    BRAVE_TAB_DRAG_CONTEXT_IMPL_PAINT_CHILDREN
     for (const ZOrderableTabContainerElement& child : orderable_children) {
       child.view()->Paint(paint_info);
     }
@@ -820,6 +822,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
         continue;
       }
 
+      BRAVE_TAB_DRAG_CONTEXT_IMPL_CALCULATE_INSERTION_INDEX
       // If there's a group header here, and we're dragging a group, we might
       // end up on either side of that header. Check both cases to find the
       // best option.
