diff --git a/chrome/browser/ui/views/tabs/tab_strip.cc b/chrome/browser/ui/views/tabs/tab_strip.cc
index 909508033c64971f073d6fd98c3cbf0488853766..e2ac7cda8eb820bb8ac0e5561f6aac81f0203a85 100644
--- a/chrome/browser/ui/views/tabs/tab_strip.cc
+++ b/chrome/browser/ui/views/tabs/tab_strip.cc
@@ -507,6 +507,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
   std::vector<gfx::Rect> CalculateBoundsForDraggedViews(
       const std::vector<raw_ptr<TabSlotView, VectorExperimental>>& views)
       override {
+    BRAVE_TAB_DRAG_CONTEXT_IMPL_CALCULATE_BOUNDS_FOR_DRAGGED_VIEWS
     DCHECK(!views.empty());
 
     std::vector<gfx::Rect> bounds;
@@ -688,6 +689,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
     // index.
     std::stable_sort(orderable_children.begin(), orderable_children.end());
 
+    BRAVE_TAB_DRAG_CONTEXT_IMPL_PAINT_CHILDREN
     for (const ZOrderableTabContainerElement& child : orderable_children)
       child.view()->Paint(paint_info);
   }
@@ -778,6 +780,7 @@ class TabStrip::TabDragContextImpl : public TabDragContext,
         continue;
       }
 
+      BRAVE_TAB_DRAG_CONTEXT_IMPL_CALCULATE_INSERTION_INDEX
       // If there's a group header here, and we're dragging a group, we might
       // end up on either side of that header. Check both cases to find the
       // best option.
