diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 561ba677ef4024e28cdf3990fb93ef538092aa0e..dcf43e2ebe1d4ca06d3588cc2c44233c1d7f6c9b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2767,6 +2767,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
