diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 1b7576ad0d04eea37668d6ed1a746074deb2c67c..4bb56fd12f86c1dd8d3834ce26377fe725a86817 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2513,6 +2513,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
