diff --git a/chrome/android/features/tab_ui/BUILD.gn b/chrome/android/features/tab_ui/BUILD.gn
index ff4b8a6542378a2597f05d40448eeb3ab5c10bd9..e089c992f94038114667e3fe8d8b79875633a37d 100644
--- a/chrome/android/features/tab_ui/BUILD.gn
+++ b/chrome/android/features/tab_ui/BUILD.gn
@@ -103,6 +103,7 @@ android_resources("java_resources") {
     "java/res/values/styles.xml",
     "java/res/xml/tabs_settings.xml",
   ]
+  sources += brave_android_features_java_resources
   deps = [
     ":java_strings_grd",
     "//chrome/android:chrome_app_java_resources",
