diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
index 5fce17414f5cce89d5dd4b68f2b3bf8f5e9f2ef4..8a90c0a2bc1521de1c8db7039c69a254e4cee587 100644
--- a/chrome/test/base/testing_profile.cc
+++ b/chrome/test/base/testing_profile.cc
@@ -414,7 +414,7 @@ void TestingProfile::Init() {
       EnsureBrowserContextKeyedServiceFactoriesBuilt();
 
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
-  if (!IsOffTheRecord()) {
+  if (!IsOffTheRecord() && !brave::IsTorProfile(GetPath())) {
     SupervisedUserSettingsService* settings_service =
         SupervisedUserSettingsServiceFactory::GetForKey(key_.get());
     TestingPrefStore* store = new TestingPrefStore();
