diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 1754dc5d96dc5101ec4309445bb21e59f083c9c7..f7765cef16b0aaff30f9192e20b86d01345cf422 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -888,6 +888,7 @@ if (!is_android) {
       "//v8",
     ]
 
+    deps += [ "//brave/test:brave_browser_tests_deps", ]
     # Runtime dependencies
     data_deps = [
       "//chrome:browser_tests_pak",
@@ -4277,6 +4278,7 @@ test("unit_tests") {
     "//ui/web_dialogs:web_dialogs_unittests",
     "//v8",
   ]
+  deps += [ "//brave/test:brave_test_support_unit", ]
 
   if (is_mac) {
     data_deps += [ "//chrome:chrome_framework" ]
