diff --git a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_unittest.cc b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_unittest.cc
index 6a2250e838812b0fc5c56df725258b1a877c6a61..59f21e0045f9fbad2399bb7068d60748419d2f39 100644
--- a/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_unittest.cc
+++ b/chrome/browser/ui/tabs/recent_tabs_sub_menu_model_unittest.cc
@@ -175,6 +175,7 @@ class RecentTabsSubMenuModelTest : public BrowserWithTestWindowTest,
     helper->VerifyExport(session_sync_service_->GetOpenTabsUIDelegate());
   }
 
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_TEST
  private:
   raw_ptr<sync_sessions::SessionSyncService, DanglingUntriaged>
       session_sync_service_;
