diff --git a/chrome/browser/ui/tabs/tab_strip_model.h b/chrome/browser/ui/tabs/tab_strip_model.h
index 763ee820c337ddcb5ed94d5dfa594f313d188ccb..b5b2e023eba95d87d27c1cb28fe28b7ff02fbdd6 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.h
+++ b/chrome/browser/ui/tabs/tab_strip_model.h
@@ -664,6 +664,7 @@ class TabStripModel : public TabGroupController {
   tabs::TabModel* GetTabForWebContents(
       const content::WebContents* contents) const;
 
+  TAB_STRIP_MODEL_H_
  private:
   FRIEND_TEST_ALL_PREFIXES(TabStripModelTest, GetIndicesClosedByCommand);
 
