diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 928334c5f92eb820faf04f4fbd58188a3115b8b8..aad3d33c687faf5e66e4cb7754668a25ded92efe 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -125,6 +125,7 @@ class AvatarToolbarButton : public ToolbarButton {
   void NotifyShowSigninPausedDelayEnded() const;
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
