diff --git a/chrome/browser/bookmarks/bookmark_model_factory.cc b/chrome/browser/bookmarks/bookmark_model_factory.cc
index e54ad07d10fef4824e1b8cebd64a12059d2a5c5c..a1d1e0540325019aaeae08660937f8814a24112c 100644
--- a/chrome/browser/bookmarks/bookmark_model_factory.cc
+++ b/chrome/browser/bookmarks/bookmark_model_factory.cc
@@ -87,6 +87,7 @@ void BookmarkModelFactory::RegisterProfilePrefs(
 
 content::BrowserContext* BookmarkModelFactory::GetBrowserContextToUse(
     content::BrowserContext* context) const {
+  BRAVE_GET_BROWSER_CONTEXT_TO_USE
   return chrome::GetBrowserContextRedirectedInIncognito(context);
 }
 
