diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index f1092fb9e3e511567eda7c11e61051e419a86d55..0470ea53d4f8a1c87d10a986a4bc6c4e9ec65b7d 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -241,6 +241,7 @@ class BrowserProcessImpl : public BrowserProcess,
   static void RegisterPrefs(PrefRegistrySimple* registry);
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
