diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
index b5dcfa93fadd12f8ac3e1de1f4b0778200c26e88..ed10f6d0c2b980f6f55b6d4b658fcb4c90b5f7a0 100644
--- a/chrome/test/base/testing_profile.cc
+++ b/chrome/test/base/testing_profile.cc
@@ -335,6 +335,7 @@ void TestingProfile::Init() {
       EnsureBrowserContextKeyedServiceFactoriesBuilt();
 
 #if BUILDFLAG(ENABLE_SUPERVISED_USERS)
+  BRAVE_INIT
   if (!IsOffTheRecord()) {
     SupervisedUserSettingsService* settings_service =
         SupervisedUserSettingsServiceFactory::GetForKey(key_.get());
