diff --git a/chrome/browser/extensions/chrome_extensions_browser_client.h b/chrome/browser/extensions/chrome_extensions_browser_client.h
index 713b97d1c08853f2bb5535b3a4ee006ba4599897..ce0336a4d1d97812fcf5ca31d55af00f1a88d99c 100644
--- a/chrome/browser/extensions/chrome_extensions_browser_client.h
+++ b/chrome/browser/extensions/chrome_extensions_browser_client.h
@@ -147,6 +147,7 @@ class ChromeExtensionsBrowserClient : public ExtensionsBrowserClient {
   std::string GetUserAgent() const override;
 
   static void set_did_chrome_update_for_testing(bool did_update);
+  BRAVE_CHROME_EXTENSIONS_BROWSER_CLIENT_H_
 
  private:
   friend struct base::LazyInstanceTraitsBase<ChromeExtensionsBrowserClient>;
