diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 59bb34fff9480fdcdd7770a2fd8d44ecde482f0d..f7ddb464142a790609b7dd4d0a376813c651189c 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4135,6 +4135,7 @@ const FeatureEntry::FeatureVariation kClayBlockingVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
