diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 0ea352c3a64afb0c1a1b24ec3fbaed0aa93166f7..76c771c835ea1b52193c4e41e9f40e7cf73e7371 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1536,6 +1536,7 @@ source_set("extensions") {
       "//components/enterprise/data_controls",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 proto_library("cws_item_service_proto") {
