diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 0268be690132128ca87c42465fdd9ef8b786ce30..c35b34fe985a22ac8dee4304f018d2ab04caff07 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1457,6 +1457,7 @@ source_set("extensions") {
       "//components/enterprise/data_controls",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 proto_library("cws_item_service_proto") {
