diff --git a/chrome/browser/ui/webui/theme_source.cc b/chrome/browser/ui/webui/theme_source.cc
index b7a75420de1a27bfdd7e3c72286f5806c23f3f9c..d73027952687aa24135235e04c2fa662f6136824 100644
--- a/chrome/browser/ui/webui/theme_source.cc
+++ b/chrome/browser/ui/webui/theme_source.cc
@@ -135,7 +135,7 @@ void ThemeSource::StartDataRequest(
   int resource_id = -1;
   if (parsed_path == "current-channel-logo") {
     switch (chrome::GetChannel()) {
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || defined(OFFICIAL_BUILD)
       case version_info::Channel::CANARY:
         resource_id = IDR_PRODUCT_LOGO_32_CANARY;
         break;
@@ -157,7 +157,10 @@ void ThemeSource::StartDataRequest(
         [[fallthrough]];
 #endif
       case version_info::Channel::UNKNOWN:
+        resource_id = IDR_PRODUCT_LOGO_32_DEVELOPMENT;
+#if 0
         resource_id = IDR_PRODUCT_LOGO_32;
+#endif
         break;
     }
   } else {
