diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index b3224f262b68f580111723cb12b9400502e1c056..be0586e9fe1e9283572bfb5c3255be1db4c6abb0 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2777,6 +2777,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
