diff --git a/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc b/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
index 826928961f1191d0d510354413439c9423a8aa15..5c0cbca98fa8c6e28aeb71b7fec227eb1e2fdbf9 100644
--- a/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
+++ b/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
@@ -158,6 +158,7 @@ PermissionPromptBubbleBaseView::PermissionPromptBubbleBaseView(
   }
 
   SetProperty(views::kElementIdentifierKey, kMainViewId);
+  BRAVE_PERMISSION_PROMPT_BUBBLE_BASE_VIEW
 }
 
 PermissionPromptBubbleBaseView::~PermissionPromptBubbleBaseView() = default;
