diff --git a/chrome/browser/download/download_item_model.h b/chrome/browser/download/download_item_model.h
index d266e2d2aaf220634f6ce2f43333d7dfbe5380c6..86ae64ae563adbbd9cd4ef215b56ad5eb3cf398e 100644
--- a/chrome/browser/download/download_item_model.h
+++ b/chrome/browser/download/download_item_model.h
@@ -145,6 +145,7 @@ class DownloadItemModel : public DownloadUIModel,
   void OnDownloadDestroyed(download::DownloadItem* download) override;
 
  private:
+  friend class BraveDownloadItemModel;
   // DownloadUIModel implementation.
   std::string GetMimeType() const override;
 
