diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 73ab1b4a9983792eee54afac0e7cd8bde3ab99dd..5f532475068c09dcb301903a6c1fdee3d397c34e 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2510,6 +2510,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
