diff --git a/chrome/browser/ui/color/BUILD.gn b/chrome/browser/ui/color/BUILD.gn
index 0dc4f47187dc7e23c3ea60e9ae70ed1c1418e675..f6c8b05955fdcc73e46bdec8ccc219e59c32b2c9 100644
--- a/chrome/browser/ui/color/BUILD.gn
+++ b/chrome/browser/ui/color/BUILD.gn
@@ -62,6 +62,7 @@ source_set("mixers") {
     "//ui/color:color",
     "//ui/color:mixers",
   ]
+  sources += brave_browser_ui_color_mixers_sources deps += brave_browser_ui_color_mixers_deps
 
   if (is_chromeos) {
     sources += [ "chromeos/native_chrome_color_mixer_chromeos.cc" ]
