diff --git a/chrome/browser/ui/views/frame/browser_frame_view_win.cc b/chrome/browser/ui/views/frame/browser_frame_view_win.cc
index 68262a642d251622380d618fc5440319816ac9f0..284a505c110603ed0aa8f358cf1a5ab92df3e364 100644
--- a/chrome/browser/ui/views/frame/browser_frame_view_win.cc
+++ b/chrome/browser/ui/views/frame/browser_frame_view_win.cc
@@ -547,6 +547,7 @@ int BrowserFrameViewWin::TopAreaHeight(bool restored) const {
                      : caption_button_container_->GetPreferredSize().height();
     return top;
   }
+  BRAVE_BROWSER_FRAME_VIEW_WIN_TOP_AREA_HEIGHT
 
   // In Refresh, the tabstrip controls its own top padding.
   if (features::IsChromeRefresh2023()) {
