diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 35ca4e5a78fa28d1f33310eebc01b97cf6b1ace5..4f92226f38298c4f5a6ba9d31664d48a2b440ba0 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -144,6 +144,7 @@ source_set("configs") {
       }
     }
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui") {
