diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 7af092135ae1d6c6b2440bba65c4475f7c409826..8bbbd36534568b5f272a4ec06a1d05cd5268ef47 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -504,6 +504,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
