diff --git a/chrome/browser/ui/tabs/tab_strip_model.h b/chrome/browser/ui/tabs/tab_strip_model.h
index 1fb046412bab997916f946837704525e4a4f9aa8..c245dbbc85f4529baff26aacb8d67fa4dc8c6cff 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.h
+++ b/chrome/browser/ui/tabs/tab_strip_model.h
@@ -657,6 +657,7 @@ class TabStripModel : public TabGroupController {
   // Returns the tab at `index` in the tabstrip.
   tabs::TabModel* GetTabAtIndex(int index) const;
 
+  TAB_STRIP_MODEL_H_
  private:
   FRIEND_TEST_ALL_PREFIXES(TabStripModelTest, GetIndicesClosedByCommand);
 
