diff --git a/chrome/android/features/tab_ui/BUILD.gn b/chrome/android/features/tab_ui/BUILD.gn
index fd2bc91569384d965b98727cbbc106d6e6268e90..8e881c0ba82857cbfe8943b323eaa5dbd29c1f2a 100644
--- a/chrome/android/features/tab_ui/BUILD.gn
+++ b/chrome/android/features/tab_ui/BUILD.gn
@@ -103,6 +103,7 @@ android_resources("java_resources") {
     "java/res/values/styles.xml",
     "java/res/xml/tabs_settings.xml",
   ]
+  sources += brave_android_features_java_resources
   deps = [
     ":java_strings_grd",
     "//chrome/android:chrome_app_java_resources",
