diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 370ea00c87c0b1864ff2b5c853c89d1b4199bbc8..4cae9bcd6eec1b61c06cc0bf2d08b3b7407c5697 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -957,6 +957,7 @@ static_library("extensions") {
     "//ui/strings",
     "//url",
   ]
+  import("//brave/browser/extensions/sources.gni") sources += brave_browser_extensions_sources deps += brave_browser_extensions_deps
 
   if (is_chromeos_ash) {
     deps += [ "//chromeos/resources:media_app_bundle_resources_grit" ]
@@ -1297,6 +1298,7 @@ static_library("extensions") {
   } else {
     sources += [ "api/braille_display_private/braille_controller_stub.cc" ]
   }
+  deps += brave_extensions_resources
 }
 
 static_library("test_support") {
