diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
index 9af242c6ad752a52a558366493538f903eaeeaca..ad4dc191a949987effb3283d7decf77cfd131a05 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
@@ -352,7 +352,7 @@ public class ToolbarManager
     }
 
     /** An {@link ObservableSupplier<Integer>} for the browser constraints of the current tab. */
-    private static class ConstraintsProxy extends ObservableSupplierImpl<Integer>
+    public static class ConstraintsProxy extends ObservableSupplierImpl<Integer>
             implements Callback<Integer> {
         private ObservableSupplier<Integer> mCurrentConstraintDelegate;
 
