diff --git a/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc b/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
index cc515b403db1484dcd65d9ac06ded8ae3538f04f..329b369ec683667dd1ca0126ea7dceeaea99f31a 100644
--- a/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
+++ b/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
@@ -158,6 +158,7 @@ void PermissionPromptBubbleBaseView::CreatePermissionButtons(
     SetButtonStyle(ui::DIALOG_BUTTON_OK, ui::ButtonStyle::kTonal);
     SetButtonStyle(ui::DIALOG_BUTTON_CANCEL, ui::ButtonStyle::kTonal);
   }
+  BRAVE_PERMISSION_PROMPT_BUBBLE_BASE_VIEW
 }
 
 void PermissionPromptBubbleBaseView::CreateExtraTextLabel(
