diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 6810625229a4bf8a6196a06294d0cb18d7013d2b..6c24889d4518418f3ba6fb9d219948da8375129a 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2799,6 +2799,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
