diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
index b64079991e74053a0856b8714db5b1a9ad0c9ab9..29ca224c199b0fd35af830decc85664050e380cd 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -669,6 +669,7 @@ void LocationBarView::Layout(PassKey) {
   if (show_overriding_permission_chip)
     text_left += text_overriding_permission_chip_indent;
 
+  BRAVE_LAYOUT_LEADING_DECORATIONS
   LocationBarLayout leading_decorations(LocationBarLayout::Position::kLeftEdge,
                                         text_left);
   LocationBarLayout trailing_decorations(
@@ -744,6 +745,7 @@ void LocationBarView::Layout(PassKey) {
     }
   };
 
+  BRAVE_LAYOUT_TRAILING_DECORATIONS
   add_trailing_decoration(page_action_icon_container_,
                           /*intra_item_padding=*/0);
   for (ContentSettingImageView* view : base::Reversed(content_setting_views_)) {
