diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 5bc1b74945a0ef14d717c8c85b6cf199b8bfc3ea..16799f9c3342c81a5118d0b3aeef23ccba72e521 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -146,6 +146,7 @@ source_set("configs") {
       }
     }
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui") {
