diff --git a/chrome/browser/profiles/profile_manager.cc b/chrome/browser/profiles/profile_manager.cc
index 4cce9ca9fbf1ded2c4b8334f8ff9be0d43e97803..2cd7eb322458a4e7b7839ab19063c9923bc05b91 100644
--- a/chrome/browser/profiles/profile_manager.cc
+++ b/chrome/browser/profiles/profile_manager.cc
@@ -609,6 +609,9 @@ void ProfileManager::CreateProfileAsync(const base::FilePath& profile_path,
       // Profile is either already in the process of being created, or new.
       // Add callback to the list.
       info->callbacks.push_back(callback);
+      if (profile_path.BaseName() == base::FilePath("Tor Profile")) {  // TODO: Find a right place for triggering this
+        static_cast<ProfileImpl*>(info->profile.get())->OnPrefsLoaded(Profile::CREATE_MODE_ASYNCHRONOUS, true);
+      }
     }
   }
 }
@@ -1503,9 +1506,11 @@ void ProfileManager::EnsureActiveProfileExistsBeforeDeletion(
   const base::FilePath last_used_profile_path =
       GetLastUsedProfileDir(user_data_dir_);
   const base::FilePath guest_profile_path = GetGuestProfilePath();
+  const base::FilePath tor_profile_path = BraveProfileManager::GetTorProfilePath();
   Profile* last_used_profile = GetProfileByPath(last_used_profile_path);
   if (last_used_profile_path != profile_dir &&
       last_used_profile_path != guest_profile_path &&
+      last_used_profile_path != tor_profile_path &&
       last_used_profile != nullptr &&
       !last_used_profile->IsLegacySupervised()) {
     FinishDeletingProfile(profile_dir, last_used_profile_path);
@@ -1518,6 +1523,7 @@ void ProfileManager::EnsureActiveProfileExistsBeforeDeletion(
     base::FilePath cur_path = profile->GetPath();
     if (cur_path != profile_dir &&
         cur_path != guest_profile_path &&
+        cur_path != tor_profile_path &&
         !profile->IsLegacySupervised() &&
         !IsProfileDirectoryMarkedForDeletion(cur_path)) {
       OnNewActiveProfileLoaded(profile_dir, cur_path, std::move(callback),
@@ -1537,6 +1543,7 @@ void ProfileManager::EnsureActiveProfileExistsBeforeDeletion(
     // legacy-supervised.
     if (cur_path != profile_dir &&
         cur_path != guest_profile_path &&
+        cur_path != tor_profile_path &&
         !entry->IsLegacySupervised() &&
         !IsProfileDirectoryMarkedForDeletion(cur_path)) {
       fallback_profile_path = cur_path;
@@ -1894,6 +1901,7 @@ void ProfileManager::ScheduleForcedEphemeralProfileForDeletion(
     base::FilePath entry_path = entry->GetPath();
     if (entry_path == profile_dir ||
         entry_path == GetGuestProfilePath() ||
+        entry_path == BraveProfileManager::GetTorProfilePath() ||
         entry->IsLegacySupervised() ||
         IsProfileDirectoryMarkedForDeletion(entry_path))
       continue;
