diff --git a/chrome/browser/resources/optimize_webui.py b/chrome/browser/resources/optimize_webui.py
index 124f748150d8952575c591eaf7deea0da76c4092..3bf518d6411b386497444e8d6433b95e7c9757ad 100755
--- a/chrome/browser/resources/optimize_webui.py
+++ b/chrome/browser/resources/optimize_webui.py
@@ -22,6 +22,8 @@ sys.path.append(os.path.join(_SRC_PATH, 'third_party', 'node'))
 import node
 import node_modules
 
+sys.path.append(os.path.join(_SRC_PATH, 'brave', 'browser', 'resources'))
+import brave_optimize_webui_config as brave_config
 
 _RESOURCES_PATH = os.path.join(_SRC_PATH, 'ui', 'webui', 'resources')
 
@@ -63,7 +65,7 @@ _VULCANIZE_BASE_ARGS = [
   '--inline-scripts',
   '--rewrite-urls-in-templates',
   '--strip-comments',
-]
+] + brave_config.vulcanize_base_args()
 
 
 _URL_MAPPINGS = [
@@ -73,7 +75,7 @@ _URL_MAPPINGS = [
     ('chrome://resources/html/', _HTML_RESOURCES_PATH),
     ('chrome://resources/js/', _JS_RESOURCES_PATH),
     ('chrome://resources/polymer/v1_0/', _POLYMER_PATH)
-]
+] + brave_config.url_mappings(_SRC_PATH)
 
 
 _VULCANIZE_REDIRECT_ARGS = list(itertools.chain.from_iterable(map(
