diff --git a/chrome/browser/devtools/devtools_ui_bindings.h b/chrome/browser/devtools/devtools_ui_bindings.h
index 04fff0694ff5d6cfee255ae4fa8121a1bb15db39..555605ed327f8ac6ebc21d455f3ffd60a3857b30 100644
--- a/chrome/browser/devtools/devtools_ui_bindings.h
+++ b/chrome/browser/devtools/devtools_ui_bindings.h
@@ -104,6 +104,8 @@ class DevToolsUIBindings : public DevToolsEmbedderMessageDispatcher::Delegate,
 
  private:
   using DevToolsUIBindingsList = std::vector<DevToolsUIBindings*>;
+  friend class BraveDevToolsUIBindings;
+  FRIEND_TEST_ALL_PREFIXES(BraveDevToolsUIBindingsBrowserTest, ThemeTest);
 
   void HandleMessageFromDevToolsFrontend(base::Value);
 
