diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
index 751c747f216a2810bec94483a61f1e1241cde201..c7973b3cb5c05a765c30f257a5214aa7293af938 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tabbed_mode/TabbedRootUiCoordinator.java
@@ -1234,7 +1234,7 @@ public class TabbedRootUiCoordinator extends RootUiCoordinator {
         mNotificationPermissionController =
                 new NotificationPermissionController(mWindowAndroid, rationaleUIDelegateSupplier);
         NotificationPermissionController.attach(mWindowAndroid, mNotificationPermissionController);
-        if (mNotificationPermissionController.requestPermissionIfNeeded(/* contextual= */ false)) {
+        if (false && mNotificationPermissionController.requestPermissionIfNeeded(/* contextual= */ false)) {
             return true;
         }
 
