diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index dcf68b763b235a292a12a153a1eba36e1759a479..ef57f32848bcda86a7f03f8b94efec6118a59c8f 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -471,6 +471,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
 #endif
     case permissions::RequestType::kIdentityProvider:
       return "identity_provider";
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
