diff --git a/components/version_info/BUILD.gn b/components/version_info/BUILD.gn
index 817b82b6936d118cb2aecbf0dbf00819378b87f3..f186156dae15580de92566b6351fca4d7c0260eb 100644
--- a/components/version_info/BUILD.gn
+++ b/components/version_info/BUILD.gn
@@ -27,6 +27,7 @@ static_library("version_info") {
     ":generate_version_info",
     "//base",
   ]
+  configs += [ "//brave/build:version" ] deps += [ "//brave/components/version_info" ]
 }
 
 # Isolate the //ui/base dependency in this target.
