diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index 4b0d071cf1bd274511a004d386def27e04af9872..34ee01eb5b77075183e76c4209a81d652045e5ea 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -822,6 +822,7 @@ void AutocompleteController::InitializeAsyncProviders(int provider_types) {
   if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
     search_provider_ = new SearchProvider(provider_client_.get(), this);
     providers_.push_back(search_provider_.get());
+      BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // Providers run in the order they're added.  Add `HistoryURLProvider` after
   // `SearchProvider` because:
@@ -1101,6 +1102,7 @@ void AutocompleteController::SortCullAndAnnotateResult(
   internal_result_.SortAndCull(input_, template_url_service_,
                                triggered_feature_service_,
                                default_match_to_preserve);
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
 
 #if DCHECK_IS_ON()
   internal_result_.Validate();
