diff --git a/components/translate/content/renderer/translate_agent.cc b/components/translate/content/renderer/translate_agent.cc
index 8079227d8f0225972ba157ce9c6bab6e0de575bc..81c3fdbeb3306568c491b87bf46243c412c6e0c5 100644
--- a/components/translate/content/renderer/translate_agent.cc
+++ b/components/translate/content/renderer/translate_agent.cc
@@ -59,7 +59,7 @@ const int kTranslateInitCheckDelayMs = 150;
 
 // The maximum number of times we'll check to see if the translate library
 // injected in the page is ready.
-const int kMaxTranslateInitCheckAttempts = 5;
+const int kMaxTranslateInitCheckAttempts = 11;
 
 // The delay we wait in milliseconds before checking whether the translation has
 // finished.
