diff --git a/components/os_crypt/os_crypt_win.cc b/components/os_crypt/os_crypt_win.cc
index 91f2c6d4f79cf233770abbbcaaedbdd246ee044e..d910172129401b00d9fc30a90b8749554c8031ae 100644
--- a/components/os_crypt/os_crypt_win.cc
+++ b/components/os_crypt/os_crypt_win.cc
@@ -177,7 +177,12 @@ bool OSCryptImpl::DecryptString16(const std::string& ciphertext,
 // static
 bool OSCryptImpl::EncryptString(const std::string& plaintext,
                             std::string* ciphertext) {
+<<<<<<< ours
   if (g_use_legacy)
+=======
+  if (IsEncryptionDisabled(plaintext, ciphertext)) { return true; }
+  if (use_legacy_)
+>>>>>>> theirs
     return EncryptStringWithDPAPI(plaintext, ciphertext);
 
   crypto::Aead aead(crypto::Aead::AES_256_GCM);
@@ -203,6 +208,7 @@ bool OSCryptImpl::EncryptString(const std::string& plaintext,
 // static
 bool OSCryptImpl::DecryptString(const std::string& ciphertext,
                             std::string* plaintext) {
+  if (IsEncryptionDisabled(ciphertext, plaintext)) { return true; }
   if (!base::StartsWith(ciphertext, kEncryptionVersionPrefix,
                         base::CompareCase::SENSITIVE))
     return DecryptStringWithDPAPI(ciphertext, plaintext);
