diff --git a/components/sync/driver/profile_sync_service.h b/components/sync/driver/profile_sync_service.h
index 3df5ee21f6ec37c5915607ec9196909c62545572..fcdb8c907709ec2ca6a1c7922bd879f3fff18b9d 100644
--- a/components/sync/driver/profile_sync_service.h
+++ b/components/sync/driver/profile_sync_service.h
@@ -258,6 +258,7 @@ class ProfileSyncService : public SyncService,
   SyncClient* GetSyncClientForTest();
 
  private:
+  friend class brave_sync::BraveProfileSyncService;
   friend class TestProfileSyncService;
 
   // Passed as an argument to StopImpl to control whether or not the sync
