diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 241d6c20050e2c599b637157799e89dfb50343cd..ed357ff5e5cd0e46e87af0d8436ebe82e1e54ad6 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -451,6 +451,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
         return "window_management";
       }
 #endif
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
