diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index 06dbf6d2331d12091ed74fa904a1f5e912eee535..780b4a44bde094e1c2112a815813b567c4330d6f 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -153,6 +153,7 @@ android_library("java") {
     "//third_party/jni_zero:jni_zero_java",
     "//ui/android:ui_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
