diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 6eba46adcc70170b5d189a9ef3dc29a0d5d0b42f..9bcc47428f8a9afce939565689c68f26b644ec62 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -310,6 +310,7 @@ CookieSettingsBase::GetCookieSettingInternal(
       IsAllowed(setting) && !is_explicit_setting && is_third_party_request &&
       block_by_global_setting &&
       !IsThirdPartyCookiesAllowedScheme(first_party_url.scheme());
+  BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL
 
   // Only set mechanism for third party allow request.
   if (IsAllowed(setting) && is_third_party_request) {
