diff --git a/components/content_settings/core/browser/content_settings_pref_provider.h b/components/content_settings/core/browser/content_settings_pref_provider.h
index 202001c1bde6ef99ea5d902bbe3abe92eb7bdd9d..307b6e37da7d040002ad6cc22022ee7fc8bf6d97 100644
--- a/components/content_settings/core/browser/content_settings_pref_provider.h
+++ b/components/content_settings/core/browser/content_settings_pref_provider.h
@@ -81,6 +81,7 @@ class PrefProvider : public UserModifiableProvider {
   ContentSettingsPref* GetPref(ContentSettingsType type) const;
 
  private:
+  friend class BravePrefProvider;
   friend class DeadlockCheckerObserver;  // For testing.
 
   void Notify(const ContentSettingsPattern& primary_pattern,
