diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index 7cfac54e5507cc69f4cce46561b81a9a2a9c049d..63c5c62e58ed3ceb9369397be2aad8daacc2572c 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -185,6 +185,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 source_set("test_support") {
