diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index 15eef22d4c87e5347c469205ab8aa05c4d387057..0bc6ddd6e2c42502a22125d711b9ccd22b5fe646 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -190,6 +190,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 source_set("test_support") {
