diff --git a/components/content_settings/core/browser/content_settings_uma_util.cc b/components/content_settings/core/browser/content_settings_uma_util.cc
index 89dd97cbd8c2d22548d628d6fe4564a87b420088..93bd5dac067ac1a595b7d27d299104cc6ba73905 100644
--- a/components/content_settings/core/browser/content_settings_uma_util.cc
+++ b/components/content_settings/core/browser/content_settings_uma_util.cc
@@ -128,6 +128,7 @@ constexpr auto kHistogramValue = base::MakeFixedFlatMap<ContentSettingsType,
 
     // As mentioned at the top, please don't forget to update ContentType in
     // enums.xml when you add entries here!
+    BRAVE_HISTOGRAM_VALUE_LIST
 });
 
 constexpr int kkHistogramValueMax =
