diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 2ea992ac5d153677bfac6afeda5ecb46efd3a3e4..09809a72e17e8de69da9536fed9d05d05c2dd6b1 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -457,6 +457,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
 #endif
     case permissions::RequestType::kIdentityProvider:
       return "identity_provider";
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
