diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index a5a186f954fa91681e469920e44d5a69fbba38ae..3d218d5a437ff0cb00e9b188b22b5a6aa8570c31 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -475,6 +475,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
 #endif
     case permissions::RequestType::kIdentityProvider:
       return "identity_provider";
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
