diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index 0c05342b04e6312bbe28b17b3eb15e63ab8867bb..85d808306856e6a505e785ceed0c588cd3a6abc2 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -304,6 +304,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
