diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index 7128e748daa6504dd9f058eb7cd4eff118a90757..3b52797c3b9ee47b202c38b16cf81616be4af4dd 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -180,6 +180,11 @@ class InfoBarDelegate {
     IOS_PERMISSIONS_INFOBAR_DELEGATE = 108,
     SUPPORTED_LINKS_INFOBAR_DELEGATE_CHROMEOS = 109,
     AUTOFILL_VIRTUAL_CARD_ENROLLMENT_INFOBAR_DELEGATE_MOBILE = 110,
+<<<<<<< ours
+=======
+    BASIC_PRINT_DEPRECATED_ACCELERATOR_DELEGATE = 111,
+    BRAVE_INFOBAR_DELEGATE_IDENTIFIERS
+>>>>>>> theirs
   };
 
   // Describes navigation events, used to decide whether infobars should be
