diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 1168fd1c3a15fd0a220bdbbc0779badab5f62073..960c6bb676a5577533dcbb5a6df1c0cd76a0be0c 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -471,6 +471,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
 #endif
     case permissions::RequestType::kIdentityProvider:
       return "identity_provider";
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
