diff --git a/components/content_settings/browser/page_specific_content_settings.cc b/components/content_settings/browser/page_specific_content_settings.cc
index edb8a7598e0aa6729e53dd9800d54a6b3c668869..f790359d58f27554339dcef66b7c125c0530664d 100644
--- a/components/content_settings/browser/page_specific_content_settings.cc
+++ b/components/content_settings/browser/page_specific_content_settings.cc
@@ -821,6 +821,7 @@ bool PageSpecificContentSettings::IsContentBlocked(
       content_type == ContentSettingsType::ADS ||
       content_type == ContentSettingsType::SOUND ||
       content_type == ContentSettingsType::CLIPBOARD_READ_WRITE ||
+      content_type == ContentSettingsType::AUTOPLAY ||
       content_type == ContentSettingsType::SENSORS ||
       content_type == ContentSettingsType::GEOLOCATION ||
       content_type == ContentSettingsType::NOTIFICATIONS) {
@@ -847,6 +848,7 @@ bool PageSpecificContentSettings::IsContentAllowed(
       content_type != ContentSettingsType::MEDIASTREAM_CAMERA &&
       content_type != ContentSettingsType::MIDI_SYSEX &&
       content_type != ContentSettingsType::CLIPBOARD_READ_WRITE &&
+      content_type != ContentSettingsType::AUTOPLAY &&
       content_type != ContentSettingsType::SENSORS &&
       content_type != ContentSettingsType::GEOLOCATION &&
       content_type != ContentSettingsType::NOTIFICATIONS) {
