diff --git a/components/content_settings/core/browser/content_settings_pref_provider.h b/components/content_settings/core/browser/content_settings_pref_provider.h
index bcecb6451b14519978464bd6bc5fcc7568f8b7a8..48a4ccc216448162f1d4b72f9c5aa9ee6fd87642 100644
--- a/components/content_settings/core/browser/content_settings_pref_provider.h
+++ b/components/content_settings/core/browser/content_settings_pref_provider.h
@@ -81,6 +81,7 @@ class PrefProvider : public UserModifiableProvider {
   ContentSettingsPref* GetPref(ContentSettingsType type) const;
 
  private:
+  friend class BravePrefProvider;
   friend class DeadlockCheckerObserver;  // For testing.
 
   void Notify(const ContentSettingsPattern& primary_pattern,
