diff --git a/components/permissions/permission_util.cc b/components/permissions/permission_util.cc
index fc66ec2efc44615aae7e80905cba8c59f6c498d9..f025afb41ab7f4835b57a0197c51b4b3808dd9bc 100644
--- a/components/permissions/permission_util.cc
+++ b/components/permissions/permission_util.cc
@@ -332,6 +332,7 @@ ContentSettingsType PermissionUtil::PermissionTypeToContentSettingTypeSafe(
       return ContentSettingsType::CAPTURED_SURFACE_CONTROL;
     case PermissionType::NUM:
       break;
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_CONTENT_SETTINGS_TYPE
   }
 
   return ContentSettingsType::DEFAULT;
