diff --git a/components/content_settings/core/browser/content_settings_default_provider.cc b/components/content_settings/core/browser/content_settings_default_provider.cc
index 1f6642baaf50f5cf1d3bd1f84a1fa9ce925599c1..44c9a5c81c06f591f3691607013b7ddd04d7d405 100644
--- a/components/content_settings/core/browser/content_settings_default_provider.cc
+++ b/components/content_settings/core/browser/content_settings_default_provider.cc
@@ -437,7 +437,12 @@ void DefaultProvider::DiscardOrMigrateObsoletePreferences() {
                                                : CONTENT_SETTING_BLOCK);
   }
   prefs_->ClearPref(kDeprecatedEnableDRM);
+<<<<<<< ours
 #endif  // BUILDFLAG(IS_CHROMEOS_ASH) || BUILDFLAG(IS_WIN)
+=======
+#endif  // BUILDFLAG(IS_CHROMEOS) || BUILDFLAG(IS_WIN)
+BRAVE_DISCARD_OR_MIGRATE_OBSOLETE_PREFERENCES
+>>>>>>> theirs
 }
 
 }  // namespace content_settings
