diff --git a/components/permissions/permission_util.cc b/components/permissions/permission_util.cc
index c869e797fe845cd8647acbdf44a9684d22e6db58..b0de7866409b720932cc717e69acf14796612952 100644
--- a/components/permissions/permission_util.cc
+++ b/components/permissions/permission_util.cc
@@ -325,6 +325,7 @@ ContentSettingsType PermissionUtil::PermissionTypeToContentSettingTypeSafe(
       return ContentSettingsType::CAPTURED_SURFACE_CONTROL;
     case PermissionType::NUM:
       break;
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_CONTENT_SETTINGS_TYPE
   }
 
   return ContentSettingsType::DEFAULT;
