diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 2d279b5967593b42a6124c98e1a4f59056075024..c53351f9a392320a6a081fbe0155c4adf960900b 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -150,6 +150,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kIdentityProvider:
       return RequestTypeForUma::PERMISSION_IDENTITY_PROVIDER;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
