diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 7ad8a26e7889dc04c084184d4f327d64747131f2..4e8214691817b53bf3ad24a64782428a94865719 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3535,6 +3535,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") deps += brave_content_browser_deps
 }
 
 if (is_android) {
