diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index f1dbc483ec57d2a1cc409b9d82448879f84ffeae..e59ac086203175e742923b45a262ee142048a097 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
