diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index b54b925f42f1008fdb5f6ee733b44d849c4d2f69..444ba0ffbb322999a193daeb2ae0cb93d441aac4 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4738,6 +4738,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
