diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 8e1115c6b2d259ca8d8fb163f6d8fa020a46ae31..ec41d0ff0797326cb2d7313caf5ef1dfa497c989 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
