diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index a9eede2eb4309cc3f164db903f6ad1514a86d67c..3213ad7545336e10980e4743fa0f9a2c32958c26 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3566,6 +3566,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") deps += brave_content_browser_deps
 }
 
 if (is_android) {
