diff --git a/content/common/frame.mojom b/content/common/frame.mojom
index ecdc75ae7888096922fb1ae1c86e9b80595d6570..c71011aad8c2fe83a9957af6ebc8cc35b3f96b78 100644
--- a/content/common/frame.mojom
+++ b/content/common/frame.mojom
@@ -572,6 +572,7 @@ struct CreateNewWindowParams {
   // Whether the opener will be suppressed in the new window, in which case
   // scripting the new window is not allowed.
   bool opener_suppressed;
+  bool no_javascript_access;
 
   // Whether new window is created for a form submission.
   bool is_form_submission;
