diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 4d169795cd87c26d2264f4b89334f50c532bc4e7..a4e891cbb77c235043e9a1aa26bee36547271044 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
