diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index f9a5d0b80ed7e7e921b545bf6be611d9a91c833a..8e39e73edde3f9179add9abd3e7d99b9e575cc11 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -879,6 +879,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
