diff --git a/content/browser/devtools/devtools_instrumentation.h b/content/browser/devtools/devtools_instrumentation.h
index 257f8f388c59bdba7664ca6dc31c6fd2432e166c..851f7bf1d12d7d358786c7386b6f9087fec7548e 100644
--- a/content/browser/devtools/devtools_instrumentation.h
+++ b/content/browser/devtools/devtools_instrumentation.h
@@ -165,6 +165,12 @@ void DidCancelPrerender(const GURL& prerendering_url,
                         FrameTreeNode* ftn,
                         PrerenderHost::FinalStatus status);
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+void FinalPageGraph(FrameTreeNode* frame_tree_node,
+                    const std::string& frame_id,
+                    const std::string& data);
+// #endif
+
 void OnSignedExchangeReceived(
     FrameTreeNode* frame_tree_node,
     absl::optional<const base::UnguessableToken> devtools_navigation_token,
