diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index c4251ab6da2535d9fd2062c5553e7f9178887a63..ab23c89ce73c19f0471eae1e6632e953cfa9de84 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -824,6 +824,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
