diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 7edafe2ef79313059c3e358c135272802c6db719..b55ea6a4fcb96ec7d9e165f03b1f9fba1946e186 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -828,6 +828,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
