diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 81447acd8daf72136b9bb5688c5d89f3277fe97d..612182a64b089be7ccbd70ebc3b682a21a6a6130 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3561,6 +3561,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") deps += brave_content_browser_deps
 }
 
 if (is_android) {
