diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
index 43fec7c9132a517ad438b55786713247de13bf63..9978efad839f92b652b891a04666720209797013 100644
--- a/content/browser/web_contents/web_contents_impl.cc
+++ b/content/browser/web_contents/web_contents_impl.cc
@@ -4665,7 +4665,7 @@ FrameTree* WebContentsImpl::CreateNewWindow(
 
   // Save the window for later if we're not suppressing the opener (since it
   // will be shown immediately).
-  if (!params.opener_suppressed) {
+  if (!params.opener_suppressed && !params.no_javascript_access) {
     if (!is_guest) {
       WebContentsView* new_view = new_contents_impl->view_.get();
 
