diff --git a/ios/chrome/browser/shared/model/prefs/BUILD.gn b/ios/chrome/browser/shared/model/prefs/BUILD.gn
index 6e0fe102b7b5e1139d3ac08ee5420fe397e27117..e6f5f68f6fb1c98bb557bd0279b98dcd89a3da45 100644
--- a/ios/chrome/browser/shared/model/prefs/BUILD.gn
+++ b/ios/chrome/browser/shared/model/prefs/BUILD.gn
@@ -127,6 +127,7 @@ source_set("browser_prefs") {
     "//ios/web/common:features",
     "//ui/base",
   ]
+  deps += [ "//brave/components/p3a:buildflags" ]
 }
 
 source_set("unit_tests") {
