diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 71ba05aa8ecdc07fc7fb1beb17aee2ece81077f5..3503001d76904035bd1672c13dd8d1e4820e834e 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -892,8 +892,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
