diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 8e5c0959f79d2ab9912026302f666c0d1fcf9b1b..c1112613b42e34411e6a7892751db509617d1ac5 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -859,8 +859,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
