diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 209ab287beac7ebb4cd2ba595d21e5a535af6949..7bab2d72b848d2ac4623c5c53dc729b5802d8070 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -776,8 +776,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
