diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 2036d3b7ca73e70bc2c375d6fe57f341332ba6a8..31c5e8c440bcdaff9988f67553bea4314cddd039 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -860,8 +860,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
