diff --git a/ios/chrome/browser/flags/BUILD.gn b/ios/chrome/browser/flags/BUILD.gn
index 6e694adb8aa613591cf6fac1dd7de9f72177e8cd..e53490d0830958ab949ded05f27b257757813811 100644
--- a/ios/chrome/browser/flags/BUILD.gn
+++ b/ios/chrome/browser/flags/BUILD.gn
@@ -104,6 +104,7 @@ source_set("flags") {
   if (ios_enable_screen_time) {
     deps += [ "//ios/chrome/browser/screen_time/model:feature_flags" ]
   }
+  deps += [ "//brave/components/brave_rewards/common/buildflags" ]
 }
 
 source_set("system_flags") {
