diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index e6c2a5fdff2d2520d584736bbe33db30c645c8c8..7cd625021c763689b1e3da78ef115c4b61e4199d 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -776,8 +776,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
