diff --git a/net/dns/host_resolver_manager.cc b/net/dns/host_resolver_manager.cc
index 48abfcd67f832429189a9f57fd6f180286738150..8db30497a953baca71e861d681877b620af05bc9 100644
--- a/net/dns/host_resolver_manager.cc
+++ b/net/dns/host_resolver_manager.cc
@@ -2782,6 +2782,7 @@ class HostResolverManager::Job : public PrioritizedDispatcher::Job,
       MaybeCacheResult(results, ttl, secure);
 
     RecordJobHistograms(results, task_type);
+    BRAVE_RECORD_DNS_COUNTS
 
     // Complete all of the requests that were attached to the job and
     // detach them.
