diff --git a/net/http/http_network_transaction.cc b/net/http/http_network_transaction.cc
index 66c41641e4c56371fb1236acd76bb241fbb55f6a..8e1b9c855a056feda65d614b4d1da90f91980fcf 100644
--- a/net/http/http_network_transaction.cc
+++ b/net/http/http_network_transaction.cc
@@ -184,7 +184,9 @@ int HttpNetworkTransaction::Start(const HttpRequestInfo* request_info,
   start_timeticks_ = base::TimeTicks::Now();
 #endif  // BUILDFLAG(ENABLE_REPORTING)
 
-  session_->GetSSLConfig(&server_ssl_config_, &proxy_ssl_config_);
+  //session_->GetSSLConfig(&server_ssl_config_, &proxy_ssl_config_);
+  //H1H2
+  session_->GetSSLConfig(&server_ssl_config_, &proxy_ssl_config_, url_.host());
 
   if (request_->load_flags & LOAD_DISABLE_CERT_NETWORK_FETCHES) {
     server_ssl_config_.disable_cert_verification_network_fetches = true;
