diff --git a/services/network/network_context.cc b/services/network/network_context.cc
index ba4281703f35be79d1aab9b37ed65288b9e13074..9e041bed4b1f2ddb91148ba1fa5ce36fe3854781 100644
--- a/services/network/network_context.cc
+++ b/services/network/network_context.cc
@@ -1900,6 +1900,7 @@ void NetworkContext::ResolveHost(
     mojo::PendingRemote<mojom::ResolveHostClient> response_client) {
   // Dns request is disallowed if network access is disabled for the nonce.
   if (network_anonymization_key.GetNonce().has_value() &&
+      host->is_scheme_host_port() &&
       !IsNetworkForNonceAndUrlAllowed(
           /*nonce=*/network_anonymization_key.GetNonce().value(),
           /*url=*/host->is_host_port_pair()
