diff --git a/third_party/blink/renderer/core/script/dynamic_module_resolver.h b/third_party/blink/renderer/core/script/dynamic_module_resolver.h
index fdd5faa1caac4422bcd5d4ddd749aadd8bb425a8..71309ac3194b082ca6f620ac66f3f81642bd06bb 100644
--- a/third_party/blink/renderer/core/script/dynamic_module_resolver.h
+++ b/third_party/blink/renderer/core/script/dynamic_module_resolver.h
@@ -11,6 +11,8 @@
 #include "third_party/blink/renderer/platform/weborigin/kurl.h"
 #include "third_party/blink/renderer/platform/wtf/text/wtf_string.h"
 
+#include "brave/third_party/blink/brave_page_graph/types.h"
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
 namespace blink {
 
 class Modulator;
@@ -33,6 +35,8 @@ class CORE_EXPORT DynamicModuleResolver final
   // Should be called w/ a valid V8 context.
   void ResolveDynamically(const ModuleRequest& module_request,
                           const ReferrerScriptInfo& referrer_info,
+                          ::brave_page_graph::ScriptId referrer_script_id,
+                          ::brave_page_graph::PageGraph* page_graph,
                           ScriptPromiseResolver*);
 
  private:
