diff --git a/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc b/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc
index 288bd85cce2fc92d35174d1730dd10470526d9fb..5dbdf0b02d77790fcfde6a33e7ea46180a82b0ee 100644
--- a/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc
+++ b/third_party/blink/renderer/core/html/canvas/canvas_async_blob_creator.cc
@@ -220,6 +220,7 @@ CanvasAsyncBlobCreator::CanvasAsyncBlobCreator(
         src_data_.reset(info, src_data_.addr(), src_data_.rowBytes());
       }
     }
+    BRAVE_CANVAS_ASYNC_BLOB_CREATOR
   }
 
   idle_task_status_ = kIdleTaskNotSupported;
