diff --git a/third_party/blink/renderer/core/loader/modulescript/worklet_module_script_fetcher.h b/third_party/blink/renderer/core/loader/modulescript/worklet_module_script_fetcher.h
index d88e7a3b68a24fca249792ac97c2159fb5875b26..0c6ea8b21dc45a898e8310f0c5f1b80715dc8f14 100644
--- a/third_party/blink/renderer/core/loader/modulescript/worklet_module_script_fetcher.h
+++ b/third_party/blink/renderer/core/loader/modulescript/worklet_module_script_fetcher.h
@@ -33,6 +33,7 @@ class CORE_EXPORT WorkletModuleScriptFetcher final
   void Fetch(FetchParameters&,
              ModuleType,
              ResourceFetcher*,
+             const Modulator*,
              ModuleGraphLevel,
              ModuleScriptFetcher::Client*) override;
 
