diff --git a/third_party/blink/renderer/core/loader/resource/image_resource.h b/third_party/blink/renderer/core/loader/resource/image_resource.h
index 2ce39712f92d47551f9ad4923774d01b3e5b4792..c507245b06e8ed295327224d5fa766a5ddd5ddb4 100644
--- a/third_party/blink/renderer/core/loader/resource/image_resource.h
+++ b/third_party/blink/renderer/core/loader/resource/image_resource.h
@@ -154,6 +154,7 @@ struct DowncastTraits<ImageResource> {
   }
 };
 
+std::string ImageDigest(Image* image);
 }  // namespace blink
 
 #endif  // THIRD_PARTY_BLINK_RENDERER_CORE_LOADER_RESOURCE_IMAGE_RESOURCE_H_
