diff --git a/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.cc b/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.cc
index c04edd826d9d54be11b96e0afd93033bfa6a5830..7009bec54a948c0be1b9b11b9b9f474d8931992b 100644
--- a/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.cc
+++ b/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.cc
@@ -21,6 +21,7 @@
 #include "third_party/blink/renderer/platform/wtf/vector.h"
 #include "v8/include/v8.h"
 
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
 // <specdef label="IMSGF"
 // href="https://html.spec.whatwg.org/C/#internal-module-script-graph-fetching-procedure">
 
@@ -157,7 +158,7 @@ void ModuleTreeLinker::FetchRoot(const KURL& original_url,
                                  ModuleType module_type,
                                  const ScriptFetchOptions& options,
                                  base::PassKey<ModuleTreeLinkerRegistry>) {
-#if DCHECK_IS_ON()
+#if 1
   original_url_ = original_url;
   module_type_ = module_type;
   root_is_inline_ = false;
@@ -172,7 +173,7 @@ void ModuleTreeLinker::FetchRoot(const KURL& original_url,
 
   KURL url = original_url;
 
-#if DCHECK_IS_ON()
+#if 1
   url_ = url;
 #endif
 
@@ -234,7 +235,7 @@ void ModuleTreeLinker::FetchRootInline(
     ModuleScript* module_script,
     base::PassKey<ModuleTreeLinkerRegistry>) {
   DCHECK(module_script);
-#if DCHECK_IS_ON()
+#if 1
   original_url_ = module_script->BaseUrl();
   url_ = original_url_;
   module_type_ = ModuleType::kJavaScript;
@@ -442,6 +443,10 @@ void ModuleTreeLinker::FetchDescendants(const ModuleScript* module_script) {
   // fetching procedure should be performed in parallel to each other.
   // ...</spec>
   for (const auto& module_request : module_requests) {
+    ::brave_page_graph::PageGraph* page_graph = modulator_->GetPageGraph();
+    if (page_graph) {
+      page_graph->RegisterModuleScriptForDescendant(module_script->SourceUrl(), module_request.url);
+    }
     // <spec step="8">... perform the internal module script graph fetching
     // procedure given moduleRequest, fetch client settings object, destination,
     // options, module script's settings object, visited set, and module
