diff --git a/third_party/blink/renderer/core/dom/events/registered_event_listener.cc b/third_party/blink/renderer/core/dom/events/registered_event_listener.cc
index baab25213958f535b2fdbe1032cf9e588bacf7dd..65ff32e89311f55baf65941cd20d0ccfeeb394cd 100644
--- a/third_party/blink/renderer/core/dom/events/registered_event_listener.cc
+++ b/third_party/blink/renderer/core/dom/events/registered_event_listener.cc
@@ -30,6 +30,8 @@
 
 namespace blink {
 
+base::AtomicSequenceNumber RegisteredEventListener::sequence_;
+
 RegisteredEventListener::RegisteredEventListener()
     : use_capture_(false),
       passive_(false),
