diff --git a/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py b/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py
index 0d7381498ac0974ea66b89e2abeea5b6e2441532..330110eb09f63c70347171b42c51026e5b0a7108 100644
--- a/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py
+++ b/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py
@@ -7596,3 +7596,4 @@ def generate_interfaces(task_queue):
                          "InitIDLInterfacesForTesting",
                          "init_idl_interfaces_for_testing",
                          for_testing=True)
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
\ No newline at end of file
