diff --git a/third_party/blink/renderer/core/script/script.h b/third_party/blink/renderer/core/script/script.h
index 59f72e6c39c4ed964b7bc49cac6cb68675c12e9c..661aff7396d07d59c0c7d012e8e331b9ebb4df6f 100644
--- a/third_party/blink/renderer/core/script/script.h
+++ b/third_party/blink/renderer/core/script/script.h
@@ -73,6 +73,13 @@ class CORE_EXPORT Script : public GarbageCollected<Script> {
   const KURL& SourceUrl() const { return source_url_; }
   const TextPosition& StartPosition() const { return start_position_; }
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  void SetParentScriptId(const int parent_script_id) {
+    parent_script_id_ = parent_script_id;
+  }
+  int ParentScriptId() const { return parent_script_id_; }
+// #endif
+
  protected:
   explicit Script(const ScriptFetchOptions& fetch_options,
                   const KURL& base_url,
@@ -110,6 +117,7 @@ class CORE_EXPORT Script : public GarbageCollected<Script> {
   const KURL source_url_;
 
   const TextPosition start_position_;
+  int parent_script_id_ = 0;
 };
 
 }  // namespace blink
