diff --git a/third_party/blink/renderer/core/loader/threadable_loader.cc b/third_party/blink/renderer/core/loader/threadable_loader.cc
index 324ede5af537ae2c5206645df6b8be46fd8a8883..b59a8f05464fafddf6adc558caabd451db988c93 100644
--- a/third_party/blink/renderer/core/loader/threadable_loader.cc
+++ b/third_party/blink/renderer/core/loader/threadable_loader.cc
@@ -154,14 +154,14 @@ void ThreadableLoader::Start(ResourceRequest request) {
   if (request_context == mojom::blink::RequestContextType::VIDEO ||
       request_context == mojom::blink::RequestContextType::AUDIO) {
     DCHECK(async);
-    RawResource::FetchMedia(params, resource_fetcher_, this);
+    raw_resource_ = RawResource::FetchMedia(params, resource_fetcher_, this);
   } else if (request_context == mojom::blink::RequestContextType::MANIFEST) {
     DCHECK(async);
-    RawResource::FetchManifest(params, resource_fetcher_, this);
+    raw_resource_ = RawResource::FetchManifest(params, resource_fetcher_, this);
   } else if (async) {
-    RawResource::Fetch(params, resource_fetcher_, this);
+    raw_resource_ = RawResource::Fetch(params, resource_fetcher_, this);
   } else {
-    RawResource::FetchSynchronously(params, resource_fetcher_, this);
+    raw_resource_ = RawResource::FetchSynchronously(params, resource_fetcher_, this);
   }
 }
 
@@ -370,7 +370,11 @@ void ThreadableLoader::DispatchDidFail(const ResourceError& error) {
   client->DidFail(resource->InspectorId(), error);
 }
 
+RawResource* ThreadableLoader::GetRawResource() const {
+  return raw_resource_;
+}
 void ThreadableLoader::Trace(Visitor* visitor) const {
+  visitor->Trace(raw_resource_);
   visitor->Trace(execution_context_);
   visitor->Trace(client_);
   visitor->Trace(resource_fetcher_);
