diff --git a/third_party/blink/renderer/core/dom/events/event_target.h b/third_party/blink/renderer/core/dom/events/event_target.h
index 29cc94646f53bb7ec5eeb2d67ebc055d6bb0408d..2289ef87d9fd88b23a39428e4d24a5f90bb2828f 100644
--- a/third_party/blink/renderer/core/dom/events/event_target.h
+++ b/third_party/blink/renderer/core/dom/events/event_target.h
@@ -87,6 +87,10 @@ class CORE_EXPORT EventTargetData final {
 
   EventListenerMap event_listener_map;
   std::unique_ptr<FiringEventIteratorVector> firing_event_iterators;
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  int last_registered_event_listener_id;
+  int GetNextRegisteredEventListenerId() const;
+// #endif
 };
 
 // All DOM event targets extend EventTarget. The spec is defined here:
