diff --git a/third_party/polymer/v3_0/BUILD.gn b/third_party/polymer/v3_0/BUILD.gn
index 4f7d02ba26d8df9f5c729e72b411eb8d37974af6..1b34818ba87bda977c7fc85fce40b7c60fae99ae 100644
--- a/third_party/polymer/v3_0/BUILD.gn
+++ b/third_party/polymer/v3_0/BUILD.gn
@@ -43,6 +43,7 @@ js_files = [
 ]
 
 if (is_chromeos_ash) {
+} if (!is_android) {
   js_files += [
     "iron-a11y-announcer/iron-a11y-announcer.js",
     "iron-dropdown/iron-dropdown.js",
@@ -169,6 +170,7 @@ ts_library("library") {
   ]
 
   if (is_chromeos_ash) {
+  } if (!is_android) {
     definitions += [
       "components-chromium/iron-a11y-announcer/iron-a11y-announcer.d.ts",
       "components-chromium/iron-dropdown/iron-dropdown.d.ts",
