diff --git a/third_party/blink/renderer/core/context_features/context_feature_settings.h b/third_party/blink/renderer/core/context_features/context_feature_settings.h
index 4dddc36f8c67dd4348a7c0fe2d8eceef135f1b9d..0387620e44f08372a90f26ac8df334cc4ea5587b 100644
--- a/third_party/blink/renderer/core/context_features/context_feature_settings.h
+++ b/third_party/blink/renderer/core/context_features/context_feature_settings.h
@@ -68,6 +68,7 @@ class CORE_EXPORT ContextFeatureSettings final
   }
 
   void Trace(Visitor*) const override;
+  BRAVE_CONTEXT_FEATURE_SETTINGS_H_
 
  private:
   bool enable_mojo_js_ = false;
