diff --git a/third_party/blink/renderer/core/frame/local_dom_window.cc b/third_party/blink/renderer/core/frame/local_dom_window.cc
index d84f42c122edee47a6804e55b988a82fa2e7abf7..887ca0f08fb0319d99d660a6e0a047a93f350e69 100644
--- a/third_party/blink/renderer/core/frame/local_dom_window.cc
+++ b/third_party/blink/renderer/core/frame/local_dom_window.cc
@@ -488,6 +488,7 @@ bool LocalDOMWindow::CanExecuteScripts(
     return false;
   }
   bool script_enabled = GetFrame()->ScriptEnabled();
+  BRAVE_LOCAL_DOM_WINDOW_CAN_EXECUTE_SCRIPTS
   if (!script_enabled && reason == kAboutToExecuteScript) {
     WebContentSettingsClient* settings_client =
         GetFrame()->GetContentSettingsClient();
