diff --git a/third_party/blink/renderer/core/loader/resource/script_resource.h b/third_party/blink/renderer/core/loader/resource/script_resource.h
index 1dd2ab1d7b9ba7f1fa183212b98f55e836d83a53..973d86dcc611b27520435ea88eb7818a76535363 100644
--- a/third_party/blink/renderer/core/loader/resource/script_resource.h
+++ b/third_party/blink/renderer/core/loader/resource/script_resource.h
@@ -38,6 +38,7 @@
 #include "third_party/blink/renderer/platform/loader/fetch/resource_loader_options.h"
 #include "third_party/blink/renderer/platform/loader/fetch/text_resource_decoder_options.h"
 
+#include <string>
 namespace blink {
 
 class FetchParameters;
@@ -259,6 +260,8 @@ struct DowncastTraits<ScriptResource> {
   }
 };
 
+std::string ScriptDigest(ScriptResource* resource);
+std::string ResourceDigest(Resource* resource);
 }  // namespace blink
 
 #endif  // THIRD_PARTY_BLINK_RENDERER_CORE_LOADER_RESOURCE_SCRIPT_RESOURCE_H_
