diff --git a/third_party/blink/renderer/core/loader/resource/image_resource_content.cc b/third_party/blink/renderer/core/loader/resource/image_resource_content.cc
index 4580c7de3629aa10c08c14061ed492ac5245d5dd..7409878c910434ad83ea1c17e46baa1ced441834 100644
--- a/third_party/blink/renderer/core/loader/resource/image_resource_content.cc
+++ b/third_party/blink/renderer/core/loader/resource/image_resource_content.cc
@@ -28,6 +28,12 @@
 #include "ui/gfx/geometry/size.h"
 #include "v8/include/v8.h"
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+namespace brave_page_graph {
+using InspectorId = uint64_t;
+}
+// #endif
+
 namespace blink {
 
 namespace {
@@ -107,6 +113,18 @@ ImageResourceContent* ImageResourceContent::Fetch(FetchParameters& params,
   return resource->GetContent();
 }
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+ImageResourceContent* ImageResourceContent::Fetch(FetchParameters& params,
+                                                  ResourceFetcher* fetcher,
+                                                  brave_page_graph::InspectorId* identifier) {
+  ImageResource* resource = ImageResource::Fetch(params, fetcher);
+  if (!resource)
+    return nullptr;
+  *identifier = resource->InspectorId();
+  return resource->GetContent();
+}
+// #endif
+
 void ImageResourceContent::SetImageResourceInfo(ImageResourceInfo* info) {
   info_ = info;
 }
