diff --git a/third_party/blink/renderer/core/inspector/inspector_dom_agent.cc b/third_party/blink/renderer/core/inspector/inspector_dom_agent.cc
index 776c83e15bb80012cc8c324d63020370821e2dc4..0ef5139a91c140402f3aa1a2a55ffb298fc266b9 100644
--- a/third_party/blink/renderer/core/inspector/inspector_dom_agent.cc
+++ b/third_party/blink/renderer/core/inspector/inspector_dom_agent.cc
@@ -98,6 +98,8 @@
 #include "third_party/blink/renderer/platform/graphics/color.h"
 #include "third_party/blink/renderer/platform/wtf/text/wtf_string.h"
 
+#include "brave/third_party/blink/brave_page_graph/page_graph.h"
+
 namespace blink {
 
 using protocol::Maybe;
@@ -2601,6 +2603,24 @@ Response InspectorDOMAgent::getFileInfo(const String& object_id, String* path) {
   return Response::Success();
 }
 
+Response InspectorDOMAgent::reportPageGraphInfo(
+    int node_id,
+    std::unique_ptr<protocol::Array<String>>* report) {
+  *report = std::make_unique<protocol::Array<String>>();
+
+  Node* const node = NodeForId(node_id);
+  if (!node) {
+    return Response::ServerError("Could not find node with given id");
+  }
+
+  ::brave_page_graph::PageGraph* const page_graph =
+      node->GetDocument().GetPageGraph();
+  if (page_graph) {
+    page_graph->GenerateReportForNode(DOMNodeIds::IdForNode(node), **report);
+  }
+
+  return Response::Success();
+}
 void InspectorDOMAgent::Trace(Visitor* visitor) const {
   visitor->Trace(dom_listeners_);
   visitor->Trace(inspected_frames_);
