diff --git a/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.h b/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.h
index 14e07ab473b5d769c136867dc0b3125a4456c254..6e9391f0e02ac1a93c8352bb25f1cb47937c96a9 100644
--- a/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.h
+++ b/third_party/blink/renderer/core/loader/modulescript/module_tree_linker.h
@@ -105,7 +105,7 @@ class CORE_EXPORT ModuleTreeLinker final : public SingleModuleClient {
 
   size_t num_incomplete_fetches_ = 0;
 
-#if DCHECK_IS_ON()
+#if DCHECK_IS_ON() // || BUILDFLAG(ENABLE_BRAVE_PAGE_GRAPH))
   KURL original_url_;
   KURL url_;
   ModuleType module_type_;
