diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index 511c5643482684b6b3bc436cd26a3f9c4ec3349c..f2021acd8abffeff065841a05155ec219235f193 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -1625,6 +1625,7 @@ ImageData* BaseRenderingContext2D::getImageData(
 }
 
 ImageData* BaseRenderingContext2D::getImageDataInternal(
+    BRAVE_GET_IMAGE_DATA_PARAMS
     int sx,
     int sy,
     int sw,
@@ -1772,6 +1773,7 @@ ImageData* BaseRenderingContext2D::getImageDataInternal(
 
   // Read pixels into |contents|.
   if (snapshot) {
+    BRAVE_GET_IMAGE_DATA
     const bool read_pixels_successful =
         snapshot->PaintImageForCurrentFrame().readPixels(
             image_info, contents.Data(), image_info.minRowBytes(), sx, sy);
