diff --git a/third_party/blink/renderer/bindings/core/v8/js_event_handler_for_content_attribute.cc b/third_party/blink/renderer/bindings/core/v8/js_event_handler_for_content_attribute.cc
index 18108a283b6740c7a4095977cf5cc07d96831e6a..74fb62e0eac09933c50972ca6c6791eaa15e8e18 100644
--- a/third_party/blink/renderer/bindings/core/v8/js_event_handler_for_content_attribute.cc
+++ b/third_party/blink/renderer/bindings/core/v8/js_event_handler_for_content_attribute.cc
@@ -12,6 +12,8 @@
 #include "third_party/blink/renderer/core/frame/local_dom_window.h"
 #include "third_party/blink/renderer/core/html/forms/html_form_element.h"
 
+#include "third_party/blink/renderer/core/probe/core_probes.h"
+
 namespace blink {
 
 JSEventHandlerForContentAttribute* JSEventHandlerForContentAttribute::Create(
@@ -234,6 +236,8 @@ v8::Local<v8::Value> JSEventHandlerForContentAttribute::GetCompiledHandler(
   // and whose callback context is settings object.
   compiled_function->SetName(V8String(isolate, function_name_));
   SetCompiledHandler(script_state_of_event_target, compiled_function);
+  probe::RegisterPageGraphScriptCompilationFromAttr(
+      &event_target, function_name_, script_body_, compiled_function);
 
   return JSEventHandler::GetListenerObject(event_target);
 }
