diff --git a/third_party/blink/renderer/core/loader/image_loader.h b/third_party/blink/renderer/core/loader/image_loader.h
index edf23b278e346c7a6f259013ce2456f77aa18ac3..7df5d336d45cb1382f199f416077dec505dbe5b8 100644
--- a/third_party/blink/renderer/core/loader/image_loader.h
+++ b/third_party/blink/renderer/core/loader/image_loader.h
@@ -36,6 +36,10 @@
 #include "third_party/blink/renderer/platform/heap/prefinalizer.h"
 #include "third_party/blink/renderer/platform/wtf/text/atomic_string.h"
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+#include "brave/third_party/blink/brave_page_graph/types.h"
+// #endif
+
 namespace blink {
 
 class ContainerNode;
@@ -259,6 +263,10 @@ class CORE_EXPORT ImageLoader : public GarbageCollected<ImageLoader>,
 
   LazyImageLoadState lazy_image_load_state_;
 
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  brave_page_graph::InspectorId inspector_id_;
+// #endif
+
   // DecodeRequest represents a single request to the Decode() function. The
   // decode requests have one of the following states:
   //
