diff --git a/third_party/blink/renderer/bindings/core/v8/scheduled_action.h b/third_party/blink/renderer/bindings/core/v8/scheduled_action.h
index e47d4a1b31bee8f783a0d58b5d7c915eb13674db..8be645d74bcdef22afde29e6a20c2aabc7406ff1 100644
--- a/third_party/blink/renderer/bindings/core/v8/scheduled_action.h
+++ b/third_party/blink/renderer/bindings/core/v8/scheduled_action.h
@@ -76,6 +76,9 @@ class ScheduledAction final : public GarbageCollected<ScheduledAction>,
   Member<V8Function> function_;
   HeapVector<ScriptValue> arguments_;
   String code_;
+// #if BUILDFLAG(BRAVE_PAGE_GRAPH_ENABLED)
+  int parent_script_id_ = 0;
+// #endif
 };
 
 }  // namespace blink
