diff --git a/third_party/blink/renderer/modules/BUILD.gn b/third_party/blink/renderer/modules/BUILD.gn
index 027cedbe5511672fa75d44258bac7f07e865d0ae..7458407e7659606e231a4bc5515a709a70e05343 100644
--- a/third_party/blink/renderer/modules/BUILD.gn
+++ b/third_party/blink/renderer/modules/BUILD.gn
@@ -19,6 +19,7 @@ if (is_ios) {
 }
 
 visibility = [ "//third_party/blink/renderer/*" ]
+visibility += brave_blink_renderer_modules_visibility
 
 config("modules_implementation") {
   defines = [ "BLINK_MODULES_IMPLEMENTATION=1" ]
@@ -175,6 +176,7 @@ component("modules") {
   # generating the snapshot for android, blink is compiled with
   # current_os="linux" and target_os="android". Using target_os is necessary as
   # we need to compile in the same way as would happen when current_os="android".
+  sub_modules += brave_blink_sub_modules
   if (target_os == "android") {
     sub_modules += [ "//third_party/blink/renderer/modules/remote_objects" ]
   } else {
