diff --git a/third_party/lit/v3_0/BUILD.gn b/third_party/lit/v3_0/BUILD.gn
index 4e9a167b4be6ac9834a9b9ff928fb9a19c962a26..8d6c98aea6835757a90692324898b9a8c648f14d 100644
--- a/third_party/lit/v3_0/BUILD.gn
+++ b/third_party/lit/v3_0/BUILD.gn
@@ -57,6 +57,7 @@ ts_library("build_ts") {
     "//ui/webui/resources/cr_components/most_visited:build_ts",
     "//ui/webui/resources/cr_components/theme_color_picker:build_ts",
     "//ui/webui/resources/cr_elements:build_ts",
+    "//chrome/browser/resources/settings:build_ts", "//brave/ui/webui/resources:build_ts"
   ]
   composite = true
   in_files = [
@@ -135,6 +136,7 @@ ts_library("build_ts") {
             "${node_modules}/@lit/reactive-element/reactive-element.d.ts",
             target_gen_dir),
   ]
+  import("//brave/third_party/lit/v3_0/sources.gni") visibility += brave_lit_build_ts_visibility
 }
 
 bundle_js("build_bundle") {
