diff --git a/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc b/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
index cc88f0e0234d7c613ab9f9bbfc003704d72bdc43..5f143f4ead8a9c829fced67a7e9f2a280f89b4c0 100644
--- a/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
+++ b/third_party/blink/renderer/core/loader/modulescript/module_script_loader.cc
@@ -146,6 +146,10 @@ void ModuleScriptLoader::FetchInternal(
     options.initiator_info.position = module_request.GetReferrerPosition();
   }
 
+  options.initiator_info.dom_node_id = module_request.Options().GetDOMNodeId();
+  options.initiator_info.parent_script_id =
+      module_request.Options().GetParentScriptId();
+
   // Note: |options| should not be modified after here.
   FetchParameters fetch_params(std::move(resource_request), options);
   fetch_params.SetModuleScript();
