diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index 9e249c7c5d262981fd2f21b9aabf909e03087e93..3c609c634e52229a3557a5dcb337ac56e95bf814 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -465,6 +465,7 @@ java_prebuilt("com_google_auto_value_auto_value_annotations_java") {
     ":*",
     "//third_party/androidx:*",
   ]
+  visibility += [ "//brave/third_party/android_deps:com_google_android_datatransport_transport_runtime_java__classes" ] supports_android = true
 }
 
 # This is generated, do not edit. Update BuildConfigGenerator.groovy instead.
