diff --git a/third_party/blink/renderer/core/probe/core_probes.json5 b/third_party/blink/renderer/core/probe/core_probes.json5
index 502f6153937bd17bb8261fced4c4b2d595a9f329..e7dfe5fa1fd56fe6a71a7a078f89d050b8865ffd 100644
--- a/third_party/blink/renderer/core/probe/core_probes.json5
+++ b/third_party/blink/renderer/core/probe/core_probes.json5
@@ -17,6 +17,37 @@
     ],
   },
   observers: {
+    PageGraphAgent: {
+      include_path: "brave/third_party/blink/brave_page_graph",
+      probes: [
+        "DidInsertDOMNode",
+        "WillRemoveDOMNode",
+        "DidCommitLoad",
+        "NodeCreated",
+        "RegisterPageGraphNodeFullyCreated",
+        "RegisterPageGraphElmForLocalScript",
+        "RegisterPageGraphElmForRemoteScript",
+        "RegisterPageGraphUrlForScriptSource",
+        "RegisterPageGraphModuleScriptForDescendant",
+        "RegisterPageGraphScriptCompilation",
+        "RegisterPageGraphModuleCompilation",
+        "RegisterPageGraphScriptCompilationFromAttr",
+        "RegisterPageGraphBindingEvent",
+        "RegisterPageGraphWebAPICallWithResult",
+        "ConsoleMessageAdded",
+        "DidModifyDOMAttr",
+        "DidRemoveDOMAttr",
+        "PrepareRequest",
+        "DidReceiveResourceResponse",
+        "DidReceiveData",
+        "DidReceiveBlob",
+        "DidFinishLoading",
+        "DidFailLoading",
+        "RegisterPageGraphEventListenerAdd",
+        "RegisterPageGraphEventListenerRemove",
+        "RegisterPageGraphRemoteFrameCreated",
+      ]
+    },
     AdTracker: {
       include_path: "third_party/blink/renderer/core/frame",
       probes: [
