diff --git a/third_party/blink/renderer/core/BUILD.gn b/third_party/blink/renderer/core/BUILD.gn
index a6ed7ad1e1a0de965ec326768ca0f5ef9bdfa87a..aba62fe9d3b3e4da46e83da54322b824bf732812 100644
--- a/third_party/blink/renderer/core/BUILD.gn
+++ b/third_party/blink/renderer/core/BUILD.gn
@@ -406,6 +406,7 @@ component("core") {
       "Carbon.framework",
     ]
   }
+  visibility += brave_blink_renderer_core_visibility public_deps += brave_blink_renderer_core_public_deps sources += brave_blink_renderer_core_sources deps += brave_blink_renderer_core_deps
 }
 
 # Certain sources should be compiled with heavy optimization enabled.
