diff --git a/third_party/blink/renderer/core/loader/link_loader.h b/third_party/blink/renderer/core/loader/link_loader.h
index 3d2215e8f4bcc3a647b533dc03b817a29239e02d..70f41dd835e1e82dbf226249cebda48452df9495 100644
--- a/third_party/blink/renderer/core/loader/link_loader.h
+++ b/third_party/blink/renderer/core/loader/link_loader.h
@@ -45,6 +45,7 @@ class ModuleScript;
 class PrerenderHandle;
 class Resource;
 class ResourceClient;
+class HTMLLinkElement;
 
 // The LinkLoader can load link rel types icon, dns-prefetch, prefetch, and
 // prerender.
@@ -64,6 +65,8 @@ class CORE_EXPORT LinkLoader final : public GarbageCollected<LinkLoader> {
 
   Resource* GetResourceForTesting();
 
+  void SetHTMLLinkElementOwner(HTMLLinkElement* link_element);
+
   void NotifyModuleLoadFinished(ModuleScript*);
   void NotifyFinished(Resource*);
 
@@ -76,6 +79,7 @@ class CORE_EXPORT LinkLoader final : public GarbageCollected<LinkLoader> {
   Member<PendingLinkPreload> pending_preload_;
 
   Member<PrerenderHandle> prerender_;
+  Member<HTMLLinkElement> link_element_;
 };
 
 }  // namespace blink
