diff --git a/ui/base/accelerators/accelerator.cc b/ui/base/accelerators/accelerator.cc
index 78a3ead2e26b599fabf006ca1c9b0688a8f93f06..927dd6fcce3d62b3fb9c57d35e376b945cf5f528 100644
--- a/ui/base/accelerators/accelerator.cc
+++ b/ui/base/accelerators/accelerator.cc
@@ -388,6 +388,7 @@ std::u16string Accelerator::ApplyLongFormModifiers(
   // Note that we use 'else-if' in order to avoid using Ctrl+Alt as a shortcut.
   // See https://devblogs.microsoft.com/oldnewthing/20040329-00/?p=40003 for
   // more information.
+  BRAVE_UI_BASE_ACCELERATOR_APPLY_LONG_FORM_MODIFIERS_
   if (IsCtrlDown())
     result = ApplyModifierToAcceleratorString(result, IDS_APP_CTRL_KEY);
   else if (IsAltDown())
