diff --git a/ui/views/widget/desktop_aura/desktop_window_tree_host_linux.cc b/ui/views/widget/desktop_aura/desktop_window_tree_host_linux.cc
index f80967fa9e0142c0a27169cb0f77ededd2604503..87179d2dfc301be3f80b68a296083276b6651d69 100644
--- a/ui/views/widget/desktop_aura/desktop_window_tree_host_linux.cc
+++ b/ui/views/widget/desktop_aura/desktop_window_tree_host_linux.cc
@@ -238,9 +238,10 @@ void DesktopWindowTreeHostLinux::DispatchEvent(ui::Event* event) {
   // or not as SendEventToSink results in copying the event and our copy of the
   // event will not set to handled unless a dispatcher or a target are
   // destroyed.
-  if (event->IsMouseEvent() && non_client_window_event_filter_) {
-    non_client_window_event_filter_->HandleMouseEventWithHitTest(
-        hit_test_code, event->AsMouseEvent());
+  if ((event->IsMouseEvent() || event->IsTouchEvent()) &&
+      non_client_window_event_filter_) {
+    non_client_window_event_filter_->HandleLocatedEventWithHitTest(
+        hit_test_code, event->AsLocatedEvent());
   }
 
   if (!event->handled())
