diff --git a/ui/webui/resources/cr_elements/BUILD.gn b/ui/webui/resources/cr_elements/BUILD.gn
index cec3cef57f30550677a257a2e1fb29617d5a3abe..ac0b8553f65fc6812ba6560b57cc89477fa7215a 100644
--- a/ui/webui/resources/cr_elements/BUILD.gn
+++ b/ui/webui/resources/cr_elements/BUILD.gn
@@ -199,6 +199,7 @@ build_webui("build") {
   ts_composite = true
   ts_deps = [ "../js:build_ts" ]
   if (include_polymer) {
+    ts_deps += ["//brave/ui/webui/resources:build_ts"]
     ts_deps += [
       "//third_party/lit/v3_0:build_ts",
       "//third_party/polymer/v3_0:library",
