diff --git a/BUILD.gn b/BUILD.gn
index b19de4038bc2b2f5d3ba6d933c6b42a67229b54f..2e291eb6fda0d207934f0a35a795a69755eca1bf 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -751,6 +751,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3151,6 +3152,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
