diff --git a/BUILD.gn b/BUILD.gn
index 4e429908762a4d7d5e5a8d19946da2d23feb4534..82921001b19d406817ca69aedf33faeaeff37af2 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -750,6 +750,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3118,6 +3119,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
