diff --git a/BUILD.gn b/BUILD.gn
index 683f4786f60c574ad050bf3dadc95cab7ac38381..28eea82ee18a6de6883b7700599a14ed585388e6 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -745,6 +745,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3068,6 +3069,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
