diff --git a/BUILD.gn b/BUILD.gn
index 7f333889e0c07caec4efb54978b73b854eafa210..6f2a22ee4eefb8cbba2d64f0ee28cfd35f9fa778 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -751,6 +751,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3151,6 +3152,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
