diff --git a/BUILD.gn b/BUILD.gn
index f30e0b522bd7699da1d25a4a072f715593a1ad38..131edcbfa274a84ec190febf1a65918ba769df44 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -744,6 +744,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3112,6 +3113,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
