diff --git a/BUILD.gn b/BUILD.gn
index e0be6c9d1d7614b0c420a2e364b047ae25b5f979..fce460fedd0b41b5664fb1d462efdfaa5f26eaa3 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -756,6 +756,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3176,6 +3177,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
