diff --git a/BUILD.gn b/BUILD.gn
index 86e2a6f55a74ae05abcba53a6b50a3fe33550be9..b60b8e90f62e7a70b911c4044b9104b9bc4cc5f7 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -745,6 +745,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3067,6 +3068,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
