diff --git a/BUILD.gn b/BUILD.gn
index 569583cb8e4e9737c449b1f65946797d503efbfc..c528648d72549e50cd6849c5e2e0be04b0b81087 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -752,6 +752,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3184,6 +3185,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
