diff --git a/include/v8-version.h b/include/v8-version.h
index c342c826957bbb7dcefe4c5087641e9ea47dcc36..614c4690bb6726458fcd8e6eb7fc654a7fbbcd89 100644
--- a/include/v8-version.h
+++ b/include/v8-version.h
@@ -11,7 +11,7 @@
 #define V8_MAJOR_VERSION 10
 #define V8_MINOR_VERSION 3
 #define V8_BUILD_NUMBER 174
-#define V8_PATCH_LEVEL 10
+#define V8_PATCH_LEVEL 11
 
 // Use 1 for candidates and 0 otherwise.
 // (Boolean macro values are not supported by all preprocessors.)
