diff --git a/include/v8-script.h b/include/v8-script.h
index b51dbc3dfd38802d2c3908310f96d46d12101ac7..102b5e1b60f2cb5b0c328d8922ac55848b22181e 100644
--- a/include/v8-script.h
+++ b/include/v8-script.h
@@ -50,6 +50,7 @@ class V8_EXPORT ScriptOrModule {
   V8_DEPRECATED("Use HostDefinedOptions")
   Local<PrimitiveArray> GetHostDefinedOptions();
   Local<Data> HostDefinedOptions();
+  //int GetScriptId() const;
 };
 
 /**
