diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 3fa602a8ed9d06fdbe19837e9e723ff77f702b52..bfbd2fadfb8770bc63fca70e4a06e2147b3ecde6 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3007,6 +3007,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
