diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index c4284321f5c43d3d6111d52af5e139039028178a..246d10faa0d31278c6a2125e55cf3169fab8ae61 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3097,6 +3097,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
