diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 40c18d75dcf69be9c86b9884e9f5deeb4e9ca7bc..9ae33ed90536b21a93cb6b115a8214031b322eae 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3004,6 +3004,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
