diff --git a/src/codegen/script-details.h b/src/codegen/script-details.h
index 5317ae989e1fb715a4c2eb59b3d244b46bcde6ac..72a2f32870972de73ff6824c9e56aab9e30c40cb 100644
--- a/src/codegen/script-details.h
+++ b/src/codegen/script-details.h
@@ -31,6 +31,7 @@ struct ScriptDetails {
   MaybeHandle<Object> source_map_url;
   MaybeHandle<Object> host_defined_options;
   REPLMode repl_mode;
+  int parent_script_id = 0;
   const ScriptOriginOptions origin_options;
 };
 
