diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 8a4b8a9a49f365fa2f6a8eac7038a8944a6ec2b9..ed6d20a7d8b7f9826cdf3d0cd54579442ee9f7a9 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3267,6 +3267,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
