diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 04e1edfb5fafd82f6c614b49241e3586c357a05f..645040002d6493e10bcf936b62de580ffe484acd 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3014,6 +3014,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
